--- data/sfs/intermediate/1987/230.txt	2012-05-23 20:41:11.000000000 +0200
+++ /tmp/tmp2OcOLU	2012-05-23 20:40:36.000000000 +0200
@@ -259,8 +259,8 @@
 maken.
 
 Domstolen får, när makarna inte varaktigt bor tillsammans, förplikta en av
-makarna att lämna bohag till den andra maken att användas av denne. Skyldig-
-heten omfattar dock bara det bohag som tillhörde makarna eller någon av dem
+makarna att lämna bohag till den andra maken att användas av denne. Skyldigheten
+omfattar dock bara det bohag som tillhörde makarna eller någon av dem
 när samlevnaden upphörde. Avtal som efter domstolens beslut ingås med tredje
 man om egendomen inskränker inte nyttjanderätten till bohaget.
 
@@ -341,8 +341,8 @@
 4. egendom som en make har ärvt och som enligt testamente av arvlåtaren
 skall vara mottagarens enskilda,
 
-5. egendom som en make erhållit genom förmånstagarförordnande vid livförsäk-
-ring, olycksfallsförsäkring, sjukförsäkring eller pensionssparande enligt
+5. egendom som en make erhållit genom förmånstagarförordnande vid livförsäkring,
+olycksfallsförsäkring, sjukförsäkring eller pensionssparande enligt
 lagen (1993:931) om individuellt pensionssparande som tecknats av någon annan
 än den andra maken med det villkoret att egendomen skall vara mottagarens
 enskilda,
@@ -419,7 +419,9 @@
 2. avhända sig, pantsätta, hyra ut eller på annat sätt med nyttjanderätt
 upplåta annan egendom som utgör makarnas gemensamma bostad, eller
 
-3. avhända sig eller pantsätta makarnas gemensamma bohag. Första stycket
+3. avhända sig eller pantsätta makarnas gemensamma bohag.
+
+Första stycket
 gäller inte bostad eller bohag som är den ena makens enskilda egendom enligt
 2 § första stycket 2--4. En make får inte heller utan den andra makens
 samtycke avhända sig, låta inteckna, hyra ut eller på annat sätt med
@@ -1248,8 +1250,8 @@
 bodelningshandingen skall anges vad en make i dessa avseenden har att
 iaktta.
 
-I mål om klander av bodelning får domstolen inhämta yttrande av bodelnings-
-förrättaren och återförvisa ärendet till honom.
+I mål om klander av bodelning får domstolen inhämta yttrande av bodelningsförrättaren
+och återförvisa ärendet till honom.
 
 9 § Bestämmelserna om särskilda rättsmedel i 58 och 59 kap.
 rättegångsbalken skall tillämpas i fråga om bodelning som förrättats
@@ -1327,8 +1329,8 @@
 inte är lagfarna domare skall få handlägga enkla ärenden enligt 17
 kap. 1 §. Lag (1996:243).
 
-5 § Uppgifter om ingångna och upplösta äktenskap registreras inom folkbok-
-föringen enligt särskilda bestämmelser. Lag (1991:495).
+5 § Uppgifter om ingångna och upplösta äktenskap registreras inom folkbokföringen
+enligt särskilda bestämmelser. Lag (1991:495).
 
 Övergångsbestämmelser
 
@@ -1364,8 +1366,8 @@
 ikraftträdandet skall göras av skattemyndigheten.
 
 3. I fråga om överklagande av sådant beslut av pastorsämbetet eller av präst
-inom svenska kyrkan om förrättande av vigsel som har meddelats före ikraft-
-trädandet gäller äldre bestämmelser.
+inom svenska kyrkan om förrättande av vigsel som har meddelats före ikraftträdandet
+gäller äldre bestämmelser.
 
 1993:130
 
@@ -1463,4 +1465,4 @@
 1. Denna lag träder i kraft den 1 oktober 2011.
 
 2. Äldre föreskrifter gäller om en ansökan eller anmälan har
-kommit in till tingsrätten före ikraftträdandet.
+kommit in till tingsrätten före ikraftträdandet.
