--- original
+++ edited
@@ -195,13 +195,15 @@ Återställd avstavning i byggnadstypstabellen, så att kolumnerna hålls isär
 		annan byggnad för produktionen utgör 
 		kraftverksbyggnad.	
 
-Industribyggnad	Byggnad som är inrättad för industriell 
+Industri-
+byggnad		Byggnad som är inrättad för industriell 
 		verksamhet och som inte utgör 
 		kraftverksbyggnad.
 
 Specialbyggnad	Med specialbyggnad avses		
 
-		Försvarsbyggnad	Byggnad som används för 
+		Försvars-
+		byggnad		Byggnad som används för 
 				försvarsändamål eller 
 				ekonomisk försvarsberedskap, 
 				om byggnaden inte är en 
