--- qngn/qi/vagrezrqvngr/bbkzy/UFI/O2580-10.kzy	1980-01-01 00:00:00.000000000 +0100
+++ /gzc/gzciKuXI8	2011-03-23 09:56:05.000000000 +0100
@@ -150,7 +150,7 @@
                 <j:fm j:iny="16" />
                 <j:fmPf j:iny="16" />
               </j:eCe>
-              <j:g>EU2010:196</j:g>
+              <j:g>EU 2010:196</j:g>
             </j:e>
             <j:e j:efvqECe="00621SP3">
               <j:eCe>
