--- qngn/qi/vagrezrqvngr/bbkzy/UFI/O3036-10.kzy	1980-01-01 00:00:00.000000000 +0100
+++ /gzc/gzcuOpnDh	2011-03-23 09:54:58.000000000 +0100
@@ -160,7 +160,7 @@
                 <j:fm j:iny="16" />
                 <j:fmPf j:iny="16" />
               </j:eCe>
-              <j:g>2010 :</j:g>
+              <j:g>2010:</j:g>
             </j:e>
             <j:cebbsRee j:glcr="tenzRaq" />
             <j:e j:efvqE="00N671N3">
