--- data/sfs/intermediate/1998/112.txt
+++ /tmp/tmp6nrl3bzr
@@ -28,7 +28,7 @@ Borttagning av bindestreck som skapats genom avstavning i 1 och 2 §§
 
 1 § Denna lag gäller elektroniska anslagstavlor. Med
 elektronisk anslagstavla avses i denna lag en tjänst för
-elektronisk förmedling av med-delanden.
+elektronisk förmedling av meddelanden.
 
 I lagen avses med meddelande text, bild, ljud eller information
 i övrigt.
@@ -43,10 +43,10 @@
 myndigheter eller inom ett företag eller en koncern,
 
 3. tjänster som skyddas av tryckfrihetsförordningen eller
-yttrandefri-hetsgrundlagen, eller
+yttrandefrihetsgrundlagen, eller
 
 4. meddelanden som är avsedda bara för en viss mottagare eller
-en be-stämd krets av mottagare (elektronisk post).
+en bestämd krets av mottagare (elektronisk post).
 
 Information till användarna
 
