--- data/sfs/intermediate/1998/1479.txt	2017-02-06 10:05:05.705307417 +0100
+++ /tmp/tmpUK9qCu	2017-02-06 10:05:00.245565000 +0100
@@ -1266,8 +1266,7 @@
 
 Sanktionsavgift
 
-17 § 
-Om Finansinspektionen har meddelat beslut om varning 
+17 § Om Finansinspektionen har meddelat beslut om varning 
 enligt 15 § andra stycket, får inspektionen besluta att 
 värdepapperscentralen eller det utsedda kreditinstitutet ska 
 betala en sanktionsavgift. Finansinspektionen får också 
@@ -1569,4 +1568,4 @@
 anta att brott enligt lagen (2005:377) om straff för 
 marknadsmissbruk vid handel med finansiella instrument har 
 begåtts gäller 8 kap. 3 § i den äldre lydelsen.
-
+
