--- data/sfs/intermediate/1998/204.txt	2009-11-11 17:02:55.750000000 +0100
+++ c:\users\staffan\appdata\local\temp\tmpqbrv6-	2009-11-11 17:01:50.024799900 +0100
@@ -509,7 +508,7 @@
 c) ändamålen med behandlingen, och
 
 d) till vilka mottagare eller kategorier av mottagare som
-uppgifterna läm-nas ut.
+uppgifterna lämnas ut.
 
 En ansökan enligt första stycket skall göras skriftligen hos
 den personuppgiftsansvarige och vara undertecknad av den
