--- data/sfs/intermediate/1998/488.txt	2009-09-26 23:44:47.932000000 +0200
+++ c:\users\staffan\appdata\local\temp\tmpwc7_ni	2009-09-26 23:43:54.055200000 +0200
@@ -188,11 +188,16 @@
 
 2. terrängvagn, och
 
-3. motorredskap klass I C för tung lastbil och personbil med
-totalvikt över 3,5 ton samt enbart ett lätt släpfordon som är
-kopplat till sådana bilar D för buss och enbart ett lätt
-släpfordon som är kopplat till buss E för släpfordon, oavsett
-vikt och antal. Lag (2009:189).
+3. motorredskap klass I
+
+C för tung lastbil och personbil med totalvikt över 3,5 ton
+samt enbart ett lätt släpfordon som är kopplat till sådana
+bilar
+
+D för buss och enbart ett lätt släpfordon som är kopplat till
+buss
+
+E för släpfordon, oavsett vikt och antal. Lag (2009:189).
 
 5 a § /Träder i kraft I:2009-10-01/
 Förarbehörigheterna A, A1 och B ger även rätt att köra
