--- data/sfs/intermediate/2009/62.txt
+++ /tmp/tmp1wmnrvzk
@@ -937,8 +937,9 @@ Avskiljande nyrad mellan 7 kap:s kapitelrubrik och dess 1 §.
 Prövningstillstånd krävs vid överklagande till kammarrätten.
 Lag (2015:274).
 
-7 kap. Ansvarsbestämmelse 1 § Till böter döms den som
-uppsåtligen eller av grov oaktsamhet
+7 kap. Ansvarsbestämmelse
+
+1 § Till böter döms den som uppsåtligen eller av grov oaktsamhet
 
 1. åsidosätter gransknings- eller uppgiftsskyldigheten enligt
 3 kap. 1 §, eller
