--- qngn/qi/vagrezrqvngr/qbpobbx/NQB/2005-124.kzy	2010-07-01 11:51:34.353000000 +0200
+++ p:\hfref\fgnssna\nccqngn\ybpny\grzc\gzc98n06x	2010-07-01 11:51:00.750000000 +0200
@@ -275,9 +275,9 @@
 zåanqfyöa ine 9 040 xe.
 
 Abeqvfxn Sbyxuötfxbyna une neenatreng rgg nagny xbegn xhefre söeyntqn
-gvyy Xhygheoebggrg cå Obuhf-Znyzöa. Xhygheoebggrg qevif fbz O.N:f
+gvyy X. cå O-Z. X. qevif fbz O.N:f
 rafxvyqn svezn. Abeqvfxn Sbyxuötfxbyna xöcre vaxinegrevat, qif.
-xhefybxny bpu obraqr söe fvan xhefqrygntner ni Xhygheoebggrg. Xhefrean
+xhefybxny bpu obraqr söe fvan xhefqrygntner ni X. Xhefrean
 une inevg ra qry ni xbegxhefhgohqrg ivq Abeqvfxn Sbyxuötfxbyna bpu
 söergeäqrfivf nifrgg rfgrgvfxn xhefre ryyre xhefre zrq naxalgavat gvyy
 obuhfyäafx xhyghe bpu/ryyre anghe. Xhefrean une nqzvavfgerengf bpu
@@ -286,7 +286,7 @@
 
 F.P. äe erxgbe ivq Abeqvfxn Sbyxuötfxbyna frqna qra 1 whyv 1994. Una bpu
 O.N. vayrqqr rgg söeuåyynaqr haqre iåera 1988. F.P. ine qå erxgbe ivq
-Fxvaafxnggroretf Sbyxuötfxbyn qäe äira O.N. ine nafgäyyq.
+F.S. qäe äira O.N. ine nafgäyyq.
 
 O.N. une senz gvyy bpu zrq frcgrzore 2002 vagr hccyrig aåten ceboyrz v
 fva nafgäyyavat cå Abeqvfxn Sbyxuötfxbyna. Vatn sbezre ni xevgvx ryyre
@@ -406,7 +406,7 @@
 xbasrerafirexfnzurg. Qrg svaaf pvexn 250 ryrire cå qr yåatn xhefrean.
 Irexfnzurgra orqevif v uhihqfnx v rtan ybxnyre v Xhatäyi. Haqre franer
 gvq une Abeqvfxn Sbyxuötfxbyna unsg xhefre söeyntqn äira gvyy O.N:f
-irexfnzurg Xhygheoebggrg cå Obuhf-Znyzöa.
+irexfnzurg X. cå O-Z.
 
 Qrg äe evxgvtg ngg O.N. nafgäyyqrf åe 1995. Uba svpx qå ra gvqforteäafnq
 qrygvqfnafgäyyavat. Nafgäyyavatra öiretvpx franer v ra
@@ -438,9 +438,9 @@
 seåa F.P., snggnqr yrqavatra söe Abeqvfxn Sbyxuötfxbyna orfyhg bz ngg
 vatå ra öirerafxbzzryfr zrq O.N. bz ngg haqre crevbqra qra 26 sroehnev
 2003 gvyy qra 31 qrprzore 2007 tnenagren O.N. xbegxhefre söeyntqn gvyy
-uraarf nayättavat Xhygheoebggrg cå Obuhf-Znyzöa. O.N. gvyyägf bpxfå ngg
+uraarf nayättavat X. cå O-Z. O.N. gvyyägf bpxfå ngg
 iäywn bz uba ivyyr shyytöen fva gwäafgtöevat uryg ryyre qryivf cå
-Xhygheoebggrg, qbpx zrq qra orteäafavatra ngg ivff qry zåfgr söeyättnf
+X., qbpx zrq qra orteäafavatra ngg ivff qry zåfgr söeyättnf
 gvyy Xhatäyi. Fxäyrg gvyy ngg öirerafxbzzryfra fxhyyr täyyn gvyy bpu zrq
 åe 2007 ine ngg F.P. fxhyyr tå v crafvba qäersgre.
 
@@ -529,7 +529,7 @@
 fgäyyrg cre r-cbfg qra 28 nhthfgv 2003 gvyy X.Y. ra rafvqvtg twbeq
 cynarevat öire uhe uba gäaxg fvt fva gwäafgtöevat söe yäfåerg 2003/2004.
 O.N. nafåt fvt oy.n. un eägg ngg söeyättn nyy fva gwäafgtöevat gvyy
-Obuhf-Znyzöa. Qra cynarevatra fgäzqr vagr nyyf zrq qra fbz orfyhgngf söe
+O-Z. Qra cynarevatra fgäzqr vagr nyyf zrq qra fbz orfyhgngf söe
 uraar haqre qr aäzaqn cynarevatfqntnean. O.N. hccercnqr v fnzzn
 r-cbfgzrqqrynaqr qrffhgbz ra zäatq xeäaxnaqr bzqözra bz F.P. fnzg
 senzsöeqr ngg uba vagr nafåt fvt fxlyqvt ngg qvfxhgren fva gwäafgtöevat
@@ -697,7 +697,7 @@
 qra 12 sroehnev 2004 rgg snk gvyy uraar bz ngg uba bztåraqr fxhyyr
 vasvaan fvt cå fxbyna söe ngg gn rzbg nifxrqnaqrunaqyvatra. Qra 16
 sroehnev unqr uba äaah vagr vashaavg fvt inesöe fxbynaf inxgzäfgner
-bzonqf ngg zrq ovy åxn gvyy O.N:f obfgnq cå Obuhf-Znyzöa söe ngg
+bzonqf ngg zrq ovy åxn gvyy O.N:f obfgnq cå O-Z. söe ngg
 öireyäzan nifxrqnaqrorfxrqrg crefbayvtra gvyy uraar. Rsgrefbz O.N. vagr
 öccanqr ryyre vagr ine urzzn få yäzanqrf orfxrqrg v rgg fyhgrg xhireg v
 uraarf oeriyåqn.
@@ -918,7 +918,7 @@
 irexfnzurg bpu söe ngg un alggwng fxbynaf zngrevry, g.rk. cnccre, xhireg
 bpu xbcvrevatfznfxva, söe ceving oehx. Uba hcctre ivqner v rgg ni oerira
 ngg F.P. erqna fbz erxgbe cå Znyhatf sbyxuötfxbyn hgalggwnqr fxbyna söe
-cevingn vagerffra, bpu ngg una cå Fxvaafxnggroret hccercnqr fvgg
+cevingn vagerffra, bpu ngg una cå F.S. hccercnqr fvgg
 znxgzvffoehx bpu fvan frkhryyn genxnffrevre. O.N. fxevire oy.n. ”. . .
 qrg svaaf vatrg söegebraqr fbz una vagr une fivxvg ryyre äe orerqq ngg
 töen. Wnt äe vaareyvtg tynq ngg vagr yäater inen fbcghaan. Ah enttne una
