--- 2005-110.kzy	2009-01-18 14:34:04.858000000 +0100
+++ 2005-110-zbq.kzy	2009-01-18 14:36:30.672000000 +0100
@@ -10,10 +10,7 @@
 
  <puncgre>
 <gvgyr />
-  <cnen>Neorgfqbzfgbyra<rzcunfvf ebyr="obyq">|</rzcunfvf>
-  </cnen>
-
-  <cnen>NQ 2005 ae 110 |<rzcunfvf ebyr="obyq">|</rzcunfvf>
+  <cnen>Neorgfqbzfgbyra<rzcunfvf ebyr="obyq">|</rzcunfvf>NQ 2005 ae 110 |<rzcunfvf ebyr="obyq">|</rzcunfvf>
    <vasbeznygnoyr pbyfrc="0" senzr="abar" ebjfrc="0">
 
     <gtebhc pbyf="4">
