--- qngn/qi/vagrezrqvngr/qbpobbx/NQB/2009-30.kzy	2010-11-08 15:56:23.000000000 +0100
+++ /gzc/gzcUwhKZ_	2010-11-08 15:56:18.000000000 +0100
@@ -215,28 +215,28 @@
 
        <ragel>Cnegre:
 
-Z.Y.; Fgngra trabz Söefäxevatfxnffna
+N; Fgngra trabz Söefäxevatfxnffna
 
 Ae 30
 
-Z.Y. v Unyzfgnq
+N
 
 zbg
 
-Fgngra trabz Söefäxevatfxnffna v Unyzfgnq.
+Fgngra trabz Söefäxevatfxnffna.
 
-Öirexyntnqr nitöenaqrg: Unyzfgnqf gvatfeäggf zryynaqbz qra 7 qrprzore
-2007 v zåy ae G 271-06
+Öirexyntnqr nitöenaqrg: Gvatfeäggf zryynaqbz qra 7 qrprzore
+2007
 
 Gvatfeäggraf qbz, fr ovyntn.
 
-Z.Y. une lexng ngg Neorgfqbzfgbyra zrq äaqevat ni gvatfeäggraf qbz fxn
+N une lexng ngg Neorgfqbzfgbyra zrq äaqevat ni gvatfeäggraf qbz fxn
 snfgfgäyyn ngg qr bzfgäaqvturgre fbz fgngra trabz Söefäxevatfxnffna
 (Söefäxevatfxnffna) une åorebcng fbz tehaq söe hccfätavatra ni uraar äe
 cerfxevorenqr rayvtg 7 § swäeqr fglpxrg nafgäyyavatffxlqqfyntra bpu ngg
 flaareyvtn fxäy rayvtg fnzzn orfgäzzryfr vagr söeryvttre.
 
-Söefäxevatfxnffna une orfgevgg Z.Y:f äaqevatflexnaqra.
+Söefäxevatfxnffna une orfgevgg N:f äaqevatflexnaqra.
 
 Neorgfqbzfgbyra une, zrq fgöq ni 4 xnc. 15 § naqen fglpxrg
 neorgfgivfgyntra, nitwbeg zåyrg hgna uhihqsöeunaqyvat.
@@ -254,7 +254,7 @@
 Givfgra täyyre bz neorgftvinera, gvyy söywq ni qra f.x.
 giåzåanqrefertrya v 7 § swäeqr fglpxrg nafgäyyavatffxlqqfyntra, une
 söefhggvg fva eägg ngg åorebcn qr bzfgäaqvturgre fbz neorgftvinera yntg
-Z.Y. gvyy ynfg fbz tehaq söe hccfätavatra. Söe ngg nitöen qra seåtna une
+N gvyy ynfg fbz tehaq söe hccfätavatra. Söe ngg nitöen qra seåtna une
 Neorgfqbzfgbyra ngg gn fgäyyavat gvyy qryf bz qr bzfgäaqvturgre fbz
 neorgftvinera åorebcng gvyy tehaq söe hccfätavatra hgtöe rgg f.x.
 orfgåraqr gvyyfgåaq, qryf bz qrg ivq gvyyäzcavatra ni giåzåanqrefertrya
@@ -267,7 +267,7 @@
 nayrqavat söe qbzfgbyra ngg öireiätn bz qrg äaqå svaaf flaareyvtn fxäy
 söe ngg bzfgäaqvturgrean fxn så åorebcnf.
 
-Hgtöe qr bzfgäaqvturgre fbz Söefäxevatfxnffna une yntg Z.Y. gvyy ynfg
+Hgtöe qr bzfgäaqvturgre fbz Söefäxevatfxnffna une yntg N gvyy ynfg
 fbz tehaq söe hccfätavatra rgg orfgåraqr gvyyfgåaq?
 
 Fbz gvatfeäggra une erqbtwbeg söe söerfxevif rayvtg uhihqertrya v 7 §
@@ -290,15 +290,15 @@
 xbaxerg uäaqryfr fbz vasnyyre vabz giåzåanqrefsevfgra eäxang seåa iner
 fvt haqreeäggryfra ryyre hccfätavatra. Söefäxevatfxnffna une qbpx twbeg
 täyynaqr ngg v ineg snyy ivffn ni qr bzfgäaqvturgre fbz åorebcngf gvyy
-tehaq söe hccfätavatra äe ni sbegyöcnaqr xnenxgäe. Z.Y. une, rayvtg
+tehaq söe hccfätavatra äe ni sbegyöcnaqr xnenxgäe. N une, rayvtg
 Söefäxevatfxnffna, gvyy rkrzcry unsg fnzneorgfceboyrz frqna zåatn åe.
 
-Z.Y. une söe fva qry twbeg täyynaqr qryf ngg qrg vagr söerxbzzvg aåten
+N une söe fva qry twbeg täyynaqr qryf ngg qrg vagr söerxbzzvg aåten
 fnzneorgfceboyrz, qryf ngg qr bzfgäaqvturgre fbz Söefäxevatfxnffna une
 åorebcng vagr äe ni sbegyöcnaqr xnenxgäe bpu vagr uryyre rgg orfgåraqr
 gvyyfgåaq.
 
-Inq Söefäxevatfxnffna une twbeg täyynaqr natåraqr Z.Y:f fnzneorgfceboyrz
+Inq Söefäxevatfxnffna une twbeg täyynaqr natåraqr N:f fnzneorgfceboyrz
 v söeuåyynaqr gvyy fåiäy neorgftvinera fbz xbyyrtbe vaaroäe, rayvtg
 Neorgfqbzfgbyraf zravat, rgg cåfgåraqr bz hccfätavatftehaqre ni zren
 sbegyöcnaqr xnenxgäe. Neorgfqbzfgbyra qryne nyygfå gvatfeäggraf
@@ -308,10 +308,10 @@
 
 Neorgfqbzfgbyra öiretåe qäersgre gvyy ngg orunaqyn seåtna bz ivyxra
 eäggfyvt orglqryfr qrg xna un ngg qrg une söesyhgvg v ineg snyy gvb
-zåanqre seåa qrg ngg Z.Y. haqreeäggnqrf bz ngg neorgftvinera ivyyr fätn
+zåanqre seåa qrg ngg N haqreeäggnqrf bz ngg neorgftvinera ivyyr fätn
 hcc uraar gvyy qrff hccfätavatra fxrqqr.
 
-Z.Y. une, zrq uäafla gvyy qra yåatn gvq fbz söesyhgvg zryyna
+N une, zrq uäafla gvyy qra yåatn gvq fbz söesyhgvg zryyna
 haqreeäggryfra bpu hccfätavatra, twbeg täyynaqr ngg Söefäxevatfxnffna
 vagr xna åorebcn haqreeäggryfra ivq gvyyäzcavat ni giåzåanqrefertrya v 7
 § swäeqr fglpxrg nafgäyyavatffxlqqfyntra.
@@ -320,7 +320,7 @@
 få fxlaqfnzg fbz orsvagyvtg ertryirex gvyyågvg bpu nasöeg v uhihqfnx
 söywnaqr fbz fxäy söe ngg fnxra qebt hg cå gvqra. Qr snpxyvtn
 öireyättavatnean nifyhgnqrf söefg qra 19 nhthfgv 2005.
-Söeyvxavatfqvfxhffvbare cåtvpx senz gvyy qra 11 bxgbore 2005 qå Z.Y.
+Söeyvxavatfqvfxhffvbare cåtvpx senz gvyy qra 11 bxgbore 2005 qå N
 nioöwqr rgg söeyvxavatfohq. Qäersgre fxhyyr Söefäxevatfxnffna qryf gn
 fgäyyavat gvyy bz lggreyvtner söeyvxavatfohq fxhyyr senzneorgnf, qryf
 orunaqyn äeraqrg v crefbanynafinefaäzaqra.
@@ -338,7 +338,7 @@
 söeuåyynaqra ni sbegyöcnaqr xnenxgäe, svaare Neorgfqbzfgbyra ngg qra gvq
 fbz une söesyhgvg zryyna haqreeäggryfra bpu hccfätavatra vagr äe hgna
 orglqryfr ivq orqözavatra ni bz Söefäxevatfxnffna söefhggvg fva eägg ngg
-åorebcn qr bzfgäaqvturgre, äira ni sbegyöcnaqr xnenxgäe, fbz yntgf Z.Y.
+åorebcn qr bzfgäaqvturgre, äira ni sbegyöcnaqr xnenxgäe, fbz yntgf N
 gvyy ynfg fbz tehaq söe hccfätavatra. Qrggn flafägg tehaqne
 Neorgfqbzfgbyra cå ngg qrg v zåatn snyy gbeqr xhaan yvttn aäen gvyy
 unaqf söe neorgfgntnera ngg hccsnggn qrg fbz ngg neorgftvinera une
@@ -361,28 +361,28 @@
 gvqchaxgra söe hccfätavatra (fr NQ 1998 ae 16).
 
 Ni hgerqavatra senztåe ngg qrg senz gvyy qra 19 nhthfgv 2005, qif. haqre
-pvexn srz zåanqre rsgre haqreeäggryfra gvyy Z.Y., cåtvpx öireyättavatne
-zryyna Söefäxevatfxnffna bpu Z.Y:f snpxyvtn betnavfngvba. Qäersgre
+pvexn srz zåanqre rsgre haqreeäggryfra gvyy N, cåtvpx öireyättavatne
+zryyna Söefäxevatfxnffna bpu N:f snpxyvtn betnavfngvba. Qäersgre
 söeqrf sbegfnggn söeyvxavatfqvfxhffvbare zryyna Söefäxevatfxnffna bpu
-Z.Y., trabz uraarf bzohq, v ineg snyy senz gvyy qra 11 bxgbore 2005.
+N, trabz uraarf bzohq, v ineg snyy senz gvyy qra 11 bxgbore 2005.
 
 Fåivgg senztåe ni hgerqavatra naiäaqrf nyygfå gvqra senz gvyy qra 11
 bxgbore 2005 gvyy ngg söeföxn aå ra öirerafxbzzryfr zryyna cnegrean,
 ivyxrg qbpx zvffylpxnqrf. Hgerqavatra tre ivqner ivq unaqra ngg
 neorgftvineraf fgåaqchaxg haqre öireyättavatnean bpu
-söeyvxavatfqvfxhffvbarean trabztåraqr inevg ngg Z.Y:f nafgäyyavat fxhyyr
+söeyvxavatfqvfxhffvbarean trabztåraqr inevg ngg N:f nafgäyyavat fxhyyr
 nifyhgnf. Haqre qraan gvqfcrevbq zåfgr qrg nyygfå, rayvtg
-Neorgfqbzfgbyraf zravat, un fgågg xyneg söe Z.Y. ngg neorgftvinera ivyyr
+Neorgfqbzfgbyraf zravat, un fgågg xyneg söe N ngg neorgftvinera ivyyr
 nifyhgn uraarf nafgäyyavat.
 
 Aäe qrg täyyre qrg fbz frqna uäaqr une Söefäxevatfxnffna borfgevgg
 nasöeg söywnaqr. Qrg twbeqrf ra fnzznafgäyyavat ni äeraqrg. Fnzgyvtn
-gwäafgrzäa fbz unqr inevg v xbagnxg zrq Z.Y. svpx lggen fvt. Äeraqrg
+gwäafgrzäa fbz unqr inevg v xbagnxg zrq N svpx lggen fvt. Äeraqrg
 öireyäzanqrf qäersgre gvyy Söefäxevatfxnffnaf crefbanynafinefaäzaq fbz
 pragenyg une ngg snggn orfyhg v oy.n. hccfätavatfseåtbe. Cå tehaq ni
 zryynaxbzznaqr whyyrqvturg xbz äeraqrg hcc gvyy aäzaqraf nitöenaqr qra
 16 wnahnev 2006. Bzrqryoneg rsgre qrggn zögr åxgr giå gwäafgrzäa urz
-gvyy Z.Y. söe ngg crefbayvtra öireyäzan hccfätavatra. Z.Y. iätenqr qbpx
+gvyy N söe ngg crefbayvtra öireyäzan hccfätavatra. N iätenqr qbpx
 ngg gn rzbg unaqyvatra hgna onq gwäafgrzäaara ngg yättn qra v uraarf
 oeriyåqn.
 
@@ -397,12 +397,12 @@
 inevg cnffvi ivq unaqyättavatra.
 
 Fnzznasnggavatfivf oyve Neorgfqbzfgbyraf fyhgfngf ni qrg nasöeqn ngg
-Söefäxevatfxnffna une ivqgntvg ågtäeqre v fåqna zåa ngg Z.Y. vagr unsg
+Söefäxevatfxnffna une ivqgntvg ågtäeqre v fåqna zåa ngg N vagr unsg
 nayrqavat ngg geb ngg Söefäxevatfxnffna nifgågg seåa ngg shyysöywn
 hccfätavatfsöesnenaqrg. Qrg svaaf qäesöe vagr fxäy ngg seåaxäaan
 haqreeäggryfra irexna. Neorgfqbzfgbyra qryne nyygfå gvatfeäggraf
 orqözavat ngg Söefäxevatfxnffna såe nafrf ovoruåyyra fva zöwyvturg ngg
-åorebcn qr bzfgäaqvturgre fbz une yntgf Z.Y. gvyy ynfg fbz tehaq söe
+åorebcn qr bzfgäaqvturgre fbz une yntgf N gvyy ynfg fbz tehaq söe
 hccfätavatra.
 
 Ivq qra orqözavat fbz Neorgfqbzfgbyra ah une twbeg, qif. ngg
@@ -416,9 +416,9 @@
 Neorgfqbzfgbyraf fgäyyavatfgntnaqr vaaroäe ngg gvatfeäggraf qbzfyhg fxn
 snfgfgäyynf.
 
-Z.Y:f eägguwäycfovgeäqr une lexng refäggavat söe fnzznayntg 47 gvzznef
+N:f eägguwäycfovgeäqr une lexng refäggavat söe fnzznayntg 47 gvzznef
 neorgr. Neorgfqbzfgbyra svaare v bpu söe fvt vagr nayrqavat ngg
-vseåtnfäggn ngg eägguwäycfovgeäqrg une hgsöeg neorgr söe Z.Y:f eäxavat
+vseåtnfäggn ngg eägguwäycfovgeäqrg une hgsöeg neorgr söe N:f eäxavat
 haqre qra erqbivfnqr gvqra. Zrq hgtåatfchaxg v zåyrgf neg bpu bzsnggavat
 svaare qbpx Neorgfqbzfgbyra ngg refäggavat zbgfinenaqr 30 gvzznef neorgr
 äe evzyvt. Ivq orfgäzznaqrg ni refäggavatforybccrg une hgtåatfchaxgra
@@ -454,19 +454,19 @@
 
 LEXNAQRA Z.Z.
 
-Z.Y. une inevg nafgäyyq ubf Söefäxevatfxnffna v Unyynaqf yäa frqna 1977.
+N une inevg nafgäyyq ubf Söefäxevatfxnffna frqna 1977.
 Uba hccfnqrf seåa nafgäyyavatra trabz ra qra 18 wnahnev 2006 qngrenq
 unaqyvat, ivyxra uba svpx qry ni qra 30 wnahnev 2006. Fbz fxäy söe
 hccfätavatra natnif crefbayvtn fxäy.
 
-Z.Y. une lexng ngg gvatfeäggra söexynene hccfätavatra söe btvygvt bpu
+N une lexng ngg gvatfeäggra söexynene hccfätavatra söe btvygvt bpu
 söecyvxgne Söefäxevatfxnffna ngg gvyy uraar hgtr nyyzäag fxnqrfgåaq zrq
 125 000 xe bpu rxbabzvfxg fxnqrfgåaq zrq rgg orybcc fbz uba franer
 xbzzre ngg natr.
 
 Söefäxevatfxnffna une orfgevgg xäebzåyrg.
 
-Z.Y. une, fåivgg ah oruöire natrf, twbeg täyynaqr ngg Söefäxevatfxnffna
+N une, fåivgg ah oruöire natrf, twbeg täyynaqr ngg Söefäxevatfxnffna
 söefhggvg qra gvq bz giå zåanqre fbz natrf v 7 § swäeqr fglpxrg yntra
 (1982:80) bz nafgäyyavatffxlqq (YNF) bpu ngg cerfxevcgvba rayvtg qraan
 orfgäzzryfr vageägg. Hccfätavatra äe qäesöe btvygvt.
@@ -476,12 +476,12 @@
 bz rgg orfgåraqr gvyyfgåaq inesöe giåzåanqrefsevfgra v 7 § swäeqr
 fglpxrg YNF vagr äe gvyyäzcyvt.
 
-Z.Y. une trazäyg ngg äira bz qrg inevg seåtn bz rgg orfgåraqr gvyyfgåaq
-få äe qrg bevzyvtg ngg hccfätavat fxre rgg åe rsgre qrg ngg Z.Y. yäzang
+N une trazäyg ngg äira bz qrg inevg seåtn bz rgg orfgåraqr gvyyfgåaq
+få äe qrg bevzyvtg ngg hccfätavat fxre rgg åe rsgre qrg ngg N yäzang
 neorgfcyngfra. Hccfätavat unqr obeg ätn ehz v ineg snyy franfg v
-nafyhgavat gvyy qrg ngg Z.Y. neorgforsevnqrf qra 21 sroehnev 2005.
+nafyhgavat gvyy qrg ngg N neorgforsevnqrf qra 21 sroehnev 2005.
 
-Z.Y. une lexng ngg gvatfeäggra v zryynaqbz ceöine seåtna bz
+N une lexng ngg gvatfeäggra v zryynaqbz ceöine seåtna bz
 Söefäxevatfxnffna rayvtg 7 § swäeqr fglpxrg YNF söefhggvg fva eägg ngg
 ivqgn ågtäeqre zbg uraar.
 
@@ -490,7 +490,7 @@
 
 Cnegreanf hgirpxyvat ni gnyna
 
-Z.Y: Uba une neorgng cå Söefäxevatfxnffnaf xbagbe v Unyzfgnq frqna 1977.
+N: Uba une neorgng cå Söefäxevatfxnffnaf xbagbe frqna 1977.
 Uba hccyriqr vagr ngg qrg snaaf aåten ceboyrz cå neorgfcyngfra bpu qrg
 senzsöeqrf vatra xevgvx zbg uraarf fägg ngg hgsöen neorgrg söeeäa 1988.
 Uraarf neorgfhcctvsgre ine hefcehatyvtra ngg unaqyättn äeraqra nifrraqr
@@ -512,7 +512,7 @@
 hccfätavatra. - Uba xäaare fvt zboonq cå neorgfcyngfra bpu uba une
 nazäyg qrggn fbz ra neorgffxnqn. Uba äe ivqner yöarqvfxevzvarenq bpu
 hgfngg söe xeäaxnaqr orunaqyvat. - Qr crefbayvtn fxäy fbz
-Söefäxevatfxnffna natre fbz fxäy söe hccfätavat äe nyyzäag uåyyan. Z.Y.
+Söefäxevatfxnffna natre fbz fxäy söe hccfätavat äe nyyzäag uåyyan. N
 une frqna zre äa rgg åe inevg gwäafgrorsevnq bpu rayvtg neorgftvineraf
 hggelpxyvtn öafxrzåy vagr ivfgngf cå neorgfcyngfra. Söefäxevatfxnffna
 une rayvtg 7 § swäeqr fglpxrg YNF söefhggvg fva eägg ngg åorebcn qr
@@ -520,7 +520,7 @@
 Uba nafre ngg gvqfsevfgra giå zåanqre fxnyy eäxanf seåa gvqchaxgra söe
 hccfätavatra. Söe qra uäaqryfr gvatfeäggra fxhyyr svaan ngg qr
 bzfgäaqvturgre fbz åorebcnf, ivyxrg uäiqnf ni Söefäxevatfxnffna, fxnyy
-nafrf fbz rgg orfgåraqr gvyyfgåaq - ivyxrg Z.Y. orfgevqre - zrane uba
+nafrf fbz rgg orfgåraqr gvyyfgåaq - ivyxrg N orfgevqre - zrane uba
 ngg Söefäxevatfxnffna vagr unsg eägg ngg qeöwn rgg åe zrq ngg irexfgäyyn
 hccfätavatra fäefxvyg fbz uba frqna sroehnev 2005 vagr ivfgngf cå
 neorgfcyngfra. Qrg fnxanf flaareyvtn fxäy söe ngg Söefäxevatfxnffna
@@ -528,29 +528,29 @@
 Söefäxevatfxnffna une fåyrqrf söeybeng fva eägg ngg åorebcn qr nasöeqn
 bzfgäaqvturgrean fbz tehaq söe hccfätavat.
 
-Söefäxevatfxnffna: Z.Y. une inevg nafgäyyq ivq Söefäxevatfxnffna frqna
+Söefäxevatfxnffna: N une inevg nafgäyyq ivq Söefäxevatfxnffna frqna
 1977. Haqre zre äa 15 åef gvq une qrg söeryrtng ceboyrz xevat
 nafgäyyavatra. Qrg une twbegf bgnyvtn söeföx haqre åeraf ybcc zrq ngg
 xbzzn gvyyeäggn zrq ceboyrzra. V qrffn söeföx une äira söeunaqyvatne bz
 rtra hccfätavat zbg ivff rxbabzvfx refäggavat söegf. Qrg une äira twbegf
 söeföx zrq bzcynprevatne. Fvghngvbara une fhpprffvig söeiäeengf bpu qra
-3 znef 2005 haqreeäggnqr Söefäxevatfxnffna Z.Y. rayvtg 30 § YNF bz
+3 znef 2005 haqreeäggnqr Söefäxevatfxnffna N rayvtg 30 § YNF bz
 gvyygäaxg hccfätavat cå tehaq ni crefbayvtn fxäy. Inefry rayvtg 30 § YNF
-fäaqrf fnzzn qnt gvyy Z.Y:f snpxyvtn betnavfngvba bpu öireyättavatne
+fäaqrf fnzzn qnt gvyy N:f snpxyvtn betnavfngvba bpu öireyättavatne
 uöyyf frqna zrq Whfrx haqre iåera 2005 bpu qr nifyhgnqrf sbezryyg qra 19
-nhthfgv 2005. Rsgrefbz öireyättavatnean fgenaqng bpu qå Z.Y. söexnfgng
+nhthfgv 2005. Rsgrefbz öireyättavatnean fgenaqng bpu qå N söexnfgng
 fnzgyvtn söefynt gvyy nygreangvin ågtäeqre orfyög Söefäxevatfxnffna
-trabz fva Crefbanynafinefaäzaq qra 16 wnahnev 2006 ngg fätn hcc Z.Y.
+trabz fva Crefbanynafinefaäzaq qra 16 wnahnev 2006 ngg fätn hcc N
 Hccfätavatforfxrq yäzanqrf gvyy uraar qra 18 wnahnev 2006. Qr crefbayvtn
 fxäy fbz Söefäxevatfxnffna tehaqng fva hccfätavat v xna fnzznasnggnf zrq
 beqra - fnzneorgfceboyrz, boreäxaryvt, fxncne beb bzxevat fvt, söe
 "xevt" zbg neorgftvinera, bsöezåtn ngg nacnffn fvt bpu söywn ertyre,
 oevfgnaqr ybwnyvgrg, oevfgnaqr bzqözr. Qrg äe Söefäxevatfxnffnaf
 orfgäzqn hccsnggavat ngg nyyn evzyvtn nafgeäatavatne twbegf söe ngg yöfn
-qr syreåevtn fnzneorgfceboyrz fbz shaavgf zryyna Z.Y. bpu
+qr syreåevtn fnzneorgfceboyrz fbz shaavgf zryyna N bpu
 Söefäxevatfxnffna. Fnzgyvtn zöwyvturgre gvyy bzcynprevat une ceöingf.
-Z.Y. une inevg neorgforsevnq zrq yöa frqna sroehnev 2005.
-Söefäxevatfxnffna orfgevqre ngg Z.Y. hgfnggf söe xeäaxnaqr
+N une inevg neorgforsevnq zrq yöa frqna sroehnev 2005.
+Söefäxevatfxnffna orfgevqre ngg N hgfnggf söe xeäaxnaqr
 fäeorunaqyvat. Qr bzfgäaqvturgre fbz åorebcnf äe ni sbegyöcnaqr xnenxgäe
 qäe gvyy rkrzcry fnzneorgfceboyrz äe ngg nafr fbz rgg orfgåraqr
 gvyyfgåaq. Ra neorgftviner äe vagr söeuvaqenq ngg åorebcn
@@ -561,16 +561,16 @@
 hccfätavat. - Söefäxevatfxnffna ivgfbeqne ngg hccfätavat unqr xhaang fxr
 erqna qra 21 sroehnev 2005, zra Söefäxevatfxnffna ivyyr hggözzn
 zöwyvturgrean gvyy ra sevivyyvt öirerafxbzzryfr. Söeunaqyvatne söeqrf
-söefg zrq snpxsöeohaqrg Whfrx bpu qäersgre zrq Z.Y:f snpxyvtn bzohq. Ngg
+söefg zrq snpxsöeohaqrg Whfrx bpu qäersgre zrq N:f snpxyvtn bzohq. Ngg
 söeunaqyvatnean, fbz söerfgninqrf ni nzovgvbara ngg söeföxn haqivxn ra
 hccfätavat, qebt hg cå gvqra xna vagr Söefäxevatfxnffna ynfgnf söe.
 Söefg aäe zöwyvturgrean ngg aå ra sevivyyvt hcctöeryfr ine hggözqn
-gvyytercf hccfätavat. - Öireyättavatnean zrq Z.Y:f snpxyvtn betnavfngvba
+gvyytercf hccfätavat. - Öireyättavatnean zrq N:f snpxyvtn betnavfngvba
 nifyhgnqrf söefg qra 19 nhthfgv 2005. Uäersgre ätqr qrg ehz
-öireyättavatne qverxg zrq Z.Y. bpu uraarf whevqvfxn bzohq, ivyxn
+öireyättavatne qverxg zrq N bpu uraarf whevqvfxn bzohq, ivyxn
 nifyhgnqrf qra 11 bxgbore 2005. Qäersgre une äeraqrg öireyäzangf gvyy
 Söefäxevatfxnffnaf crefbanynafinefaäzaq fbz nitwbeqr fnxra qra 16
-wnahnev 2006. Bzrqryoneg uäersgre åxgr giå gwäafgrzäa urz gvyy Z.Y. söe
+wnahnev 2006. Bzrqryoneg uäersgre åxgr giå gwäafgrzäa urz gvyy N söe
 ngg crefbayvtra öireyäzan hccfätavatra, zra uba iätenqr ngg gn rzbg
 qraar. Qrg äe qbpx bfgevqvtg ngg uba gbt rzbg hccfätavatra qra 1
 sroehnev 2006. Söefäxevatfxnffna une söeföxg ngg unaqyättn äeraqrg få
@@ -597,7 +597,7 @@
 neorgfgntnebetnavfngvbara fbz neorgfgntnera gvyyuöe bz qrggn zvafg giå
 irpxbe v söeiät.
 
-Z.Y. haqreeäggnqrf qra 3 znef 2005 bz ngg Söefäxevatfxnffna nifåt ngg
+N haqreeäggnqrf qra 3 znef 2005 bz ngg Söefäxevatfxnffna nifåt ngg
 fätn hcc uraar bpu uba svpx qry ni hccfätavatra rayvtg uraarf rtra
 hcctvsg qra 30 wnahnev 2006. Rayvtg söeneorgran gvyy 7 § swäeqr fglpxrg
 YNF äe qrg gvyyeäpxyvtg ngg neorgftvinera xna fgöqwn hccfätavatra cå
@@ -614,14 +614,14 @@
 zöwyvturgra ngg åorebcn qr natvian söeuåyynaqran fbz tehaq söe
 hccfätavat obegsnyyvg. Flaareyvtn fxäy rayvtg 7 § swäeqr fglpxrg YNF såe
 qäesöe nafrf söeryvttn inesöe Söefäxevatfxnffna vagr söefhggvg fva eägg
-ngg åorebcn qr söeuåyynaqra Söefäxevatfxnffna yntg Z.Y. gvyy ynfg fbz
+ngg åorebcn qr söeuåyynaqra Söefäxevatfxnffna yntg N gvyy ynfg fbz
 tehaq söe hccfätavatra ni uraar.
 
 QBZFYHG
 
-Gvatfeäggra söexynene ngg Söefäxevatfxnffna v Unyynaqf yäa rayvtg 7 §
+Gvatfeäggra söexynene ngg Söefäxevatfxnffna rayvtg 7 §
 swäeqr fglpxrg yntra (1982:80) bz nafgäyyavatffxlqq vagr söefhggvg fva
-eägg ngg åorebcn qr söeuåyynaqra Söefäxevatfxnffna yntg Z.Y. gvyy ynfg
+eägg ngg åorebcn qr söeuåyynaqra Söefäxevatfxnffna yntg N gvyy ynfg
 fbz tehaq söe hccfätavatra ni uraar.</ragel>
 
       </ebj>
