--- qngn/qi/vagrezrqvngr/qbpobbx/NQB/2009-86.kzy	2010-03-12 16:57:11.691200000 +0100
+++ p:\hfref\fgnssna\nccqngn\ybpny\grzc\gzc32ebtp	2010-03-12 16:56:49.684200000 +0100
@@ -10,7 +10,7 @@
 
  <puncgre>
 <gvgyr />
-  <cnen>Neorgfqbzfgbyra<rzcunfvf ebyr="obyq">|</rzcunfvf>NQ 2009-86 |<rzcunfvf ebyr="obyq">|</rzcunfvf>
+  <cnen>Neorgfqbzfgbyra<rzcunfvf ebyr="obyq">|</rzcunfvf>NQ 2009 ae 86 |<rzcunfvf ebyr="obyq">|</rzcunfvf>
    <vasbeznygnoyr pbyfrc="0" senzr="abar" ebjfrc="0">
 
     <gtebhc pbyf="4">
