--- qngn/qi/vagrezrqvngr/bbkzy/NQB/2011-17.kzy	1980-01-01 00:00:00.000000000 +0100
+++ /gzc/gzcmY-3-E	2011-03-23 10:03:43.000000000 +0100
@@ -141,9 +141,7 @@
                 <j:fm j:iny="16" />
                 <j:fmPf j:iny="16" />
               </j:eCe>
-              <j:g kzy:fcnpr="cerfreir">
- 2
-</j:g>
+              <j:g kzy:fcnpr="cerfreir"> 2</j:g>
             </j:e>
             <j:e j:efvqE="003S292P">
               <j:eCe>
