--- data/sfs/intermediate/1999/1229.txt.bz2
+++ 
@@ -15176,6 +15177,7 @@ [version: 2017:1279-] Korrigerad formattering av strecksatslistor i 44:12-13, styckeindelning i 44:21, paragrafindelning i 48:26 samt tabell i 63:3
 underliggande tillgångar eller av kursindex eller liknande.
 
 12 § Med option avses en rätt för innehavaren att
+
 - förvärva eller avyttra aktier, obligationer eller andra
 tillgångar till ett bestämt pris, eller
 
@@ -15187,6 +15189,7 @@
 Huvudregel
 
 13 § Kapitalvinsten ska beräknas som skillnaden mellan
+
 - ersättningen för den avyttrade tillgången eller för den
 utfärdade förpliktelsen minskad med utgifterna för avyttringen
 eller utfärdandet, och
@@ -15255,6 +15258,7 @@
 avyttrats vid tidpunkten för utnyttjandet. Lag (2007:1419).
 
 Tillgångar som förvärvas genom arv, gåva m.m.
+
 21 § Om en tillgång förvärvas genom arv, testamente, gåva,
 bodelning eller på liknande sätt, inträder förvärvaren i den
 tidigare ägarens skattemässiga situation.
@@ -16901,11 +16906,13 @@
 som sägs om en tillgång i 26 och 27 §§ ska tillämpas också på
 en förpliktelse med sådan underliggande tillgång. Lag
 (2007:1419).
+
 26 § Juridiska personers kapitalförluster på delägarrätter för
 vilka avdrag ska göras får dras av bara mot kapitalvinster på
 delägarrätter. Om en del av en sådan förlust inte kan dras av,
 får den dras av mot en annan juridisk persons kapitalvinster
 på delägarrätter om var och en av de juridiska personerna
+
 1. begär det för ett beskattningsår som har samma
 deklarationstidpunkt enligt skatteförfarandelagen (2011:1244)
 eller som skulle ha haft det om inte någon av de juridiska
@@ -20613,15 +20620,19 @@
                                 procent av det belopp med vilket
                                 den fastställda förvärvsinkomsten
                                 överstiger 0,99 prisbasbelopp
+
 överstiger 2,72 men inte
 3,11 prisbasbelopp              0,77 prisbasbelopp
+
 överstiger 3,11 men inte
 7,88 prisbasbelopp              0,77 prisbasbelopp minskat med
                                 10 procent av det belopp med
                                 vilket den fastställda
                                 förvärvsinkomsten överstiger
                                 3,11 prisbasbelopp
+
 överstiger 7,88 prisbasbelopp   0,293 prisbasbelopp
+
 Lag (2011:1256).
 
 3 a § För dem som vid beskattningsårets ingång har fyllt 65 
@@ -20675,6 +20686,7 @@
 
 överstiger 12,43 
 prisbasbelopp			0,422 prisbasbelopp	
+
 Lag (2017:1212).
 
 4 § För dem som är obegränsat skattskyldiga bara under en del
