--- data/sfs/intermediate/1999/1229.txt.bz2
+++ 
@@ -15176,6 +15177,7 @@ [version: 2017:1279-] Korrigerad formattering av strecksatslistor i 44:12-13, styckeindelning i 44:21, paragrafindelning i 48:26 samt tabell i 63:3
 underliggande tillgångar eller av kursindex eller liknande.
 
 12 § Med option avses en rätt för innehavaren att
+
 - förvärva eller avyttra aktier, obligationer eller andra
 tillgångar till ett bestämt pris, eller
 
@@ -15187,6 +15189,7 @@
 Huvudregel
 
 13 § Kapitalvinsten ska beräknas som skillnaden mellan
+
 - ersättningen för den avyttrade tillgången eller för den
 utfärdade förpliktelsen minskad med utgifterna för avyttringen
 eller utfärdandet, och
