--- qngn/qi/vagrezrqvngr/bbkzy/UIF/O4891-09.kzy	1980-01-01 00:00:00.000000000 +0100
+++ /gzc/gzcjrOLAw	2011-03-23 09:53:45.000000000 +0100
@@ -142,7 +142,7 @@
                 <j:fm j:iny="16" />
                 <j:fmPf j:iny="16" />
               </j:eCe>
-              <j:g>EU2010:113</j:g>
+              <j:g>EU 2010:113</j:g>
             </j:e>
             <j:e j:efvqECe="00621SP3">
               <j:eCe>
