--- qngn/qi/vagrezrqvngr/bbkzy/USQ/4337-12.kzy	1980-01-01 00:00:00.000000000 +0100
+++ /gzc/gzc9dwXeD	2016-10-02 22:33:16.407977000 +0200
@@ -329,7 +329,7 @@
                 <j:fm j:iny="16" />
                 <j:fmPf j:iny="16" />
               </j:eCe>
-              <j:g>4337-12</j:g>
+              <j:g>0000-12</j:g>
             </j:e>
             <j:e j:efvqECe="00494O59">
               <j:eCe>
