--- data/sfs/intermediate/1997/192.txt	2017-02-06 10:00:11.591223305 +0100
+++ /tmp/tmpNVMOA5	2017-02-06 09:59:35.216944000 +0100
@@ -60,7 +60,6 @@
 styrelse och revisorer samt stadgar som innebär att
 sammanslutningen är öppen. Lag (2009:289).
 
-6 a § 
 6 a § En sammanslutning, som har auktorisation enligt 6 §, 
 får beviljas auktorisation att arbeta med internationell 
 adoptionsförmedling i ett annat land, i viss del av ett annat 
@@ -276,4 +275,4 @@
 ut. Föreskrifterna i 5 och 8 c §§ skall dock tillämpas från
 ikraftträdandet av denna lag.
 
-
+
