--- qngn/qi/vagrezrqvngr/bbkzy/ERT/5486-09.kzy	1980-01-01 00:00:00.000000000 +0100
+++ /gzc/gzcXniZkD	2011-03-23 09:50:53.000000000 +0100
@@ -143,7 +143,7 @@
                 <j:fm j:iny="16" />
                 <j:fmPf j:iny="16" />
               </j:eCe>
-              <j:g>EÅ 2010 ers.117</j:g>
+              <j:g>EÅ 2010 ers. 117</j:g>
             </j:e>
             <j:e j:efvqECe="00621SP3">
               <j:eCe>
