--- data/sfs/intermediate/2008/1272.txt	2009-11-11 17:21:25.778000000 +0100
+++ c:\users\staffan\appdata\local\temp\tmpn8tsk9	2009-11-11 17:20:32.054200000 +0100
@@ -16,7 +16,7 @@
 
 1 § Denna förordning innehåller föreskrifter om sådant stöd som
 Swedfund International AB (Swedfund) får pröva enligt lagen
-(2008:000) om överlämnande av vissa förvaltningsuppgifter till
+(2008:1271) om överlämnande av vissa förvaltningsuppgifter till
 Swedfund International AB.
 
 Definitioner
