--- data/sfs/intermediate/2005/551.txt.bz2
+++ 
@@ -11956,6 +11956,7 @@ Korrigerat styckeindelning i 29:8
 biträtt ett förslag om att väcka en skadeståndstalan eller, när
 det gäller en styrelseledamot eller den verkställande
 direktören, har röstat mot ett förslag om ansvarsfrihet.
+
 8 § En uppgörelse i fråga om skadeståndsskyldighet till bolaget
 enligt 1-3 §§ får träffas endast av bolagsstämman och endast
 under förutsättning att inte ägare till minst en tiondel av
