--- data/sfs/intermediate/1962/700.txt.bz2
+++ 
@@ -528,8 +528,8 @@ [version: 2018:1310-2019:310] Formatteringsfix vid 4 kap. 4 b § 
 han bor eller har bott tillsammans med under
 äktenskapsliknande förhållanden, ska han i stället dömas för
 grov kvinnofridskränkning till samma straff. Lag (2013:367).
- 
+
 4 b § /Upphör att gälla U:2021-05-01/
 Den som förföljer en person genom brottsliga 
 gärningar som utgör
 
