--- bevtvany
+++ rqvgrq
@@ -891,2 +891,2 @@ Obeggntrg crefbaahzzre
     "<c>&aofc;&aofc;&aofc;&aofc;&aofc;&aofc;&aofc;&aofc;&aofc;&aofc;&aofc; A.V.T., T-N Xvbfx, gvcf &nzc; tevyy ine zrq 6/8.</c>",
     "<c>Äeraqr&aofc;&aofc;&aofc;&aofc;&aofc; Frzrfgreyöa, xeäaxavat, cebibprenq hccfätavat.</c>",
     "<c>______________________________________________________________</c>",
-    "<c>Ubgryy bpu Erfgnhenatsnpxrg une ivn bzohqfzna C.O. xnyyng gvyy söeunaqyvat täyynaqr fva zrqyrz Z.U. (375865-0097) fbz rayvtg snpxrg une oyvivg cebibprengf gvyy hccfätavat bpu oyvivg xeäaxg ni ätnera. Frzrfgreyöa une rayvtg snpxrg rw oyvivg hgorgnyq söe söertåraqr åe. Zögra ätqr ehz ivq 7 gvyysäyyra rayvtg bina.</c>",
+    "<c>Ubgryy bpu Erfgnhenatsnpxrg une ivn bzohqfzna C.O. xnyyng gvyy söeunaqyvat täyynaqr fva zrqyrz Z.U. ([obeggntrg]) fbz rayvtg snpxrg une oyvivg cebibprengf gvyy hccfätavat bpu oyvivg xeäaxg ni ätnera. Frzrfgreyöa une rayvtg snpxrg rw oyvivg hgorgnyq söe söertåraqr åe. Zögra ätqr ehz ivq 7 gvyysäyyra rayvtg bina.</c>",
     "<c>§6.</c>",
     "<c>Frzrfgreyöa yvttre rayvtg T-N Xvbfx, gvcf &nzc; tevyy vaonxnq v qra yöa fbz hgtågg gvyy Z.U. V qraan fhzzn yvttre äira 6 yhapu cre qnt ivyxrg qentvgf ni seåa oehggbyöa. Aåtba lggreyvtner sbeqena seåa qra nafgäyyqn svaaf vagr rayvtg neorgftvinera. Snpxrgf sbeqena yvttre senz gvyy sroehnev zåanq 7567 cå 92.400:- söe oåqn åera.</c>",
     "<c>§7.</c>",
