--- bevtvany
+++ rqvgrq
@@ -605,3 +605,3 @@ Obeggntan gryrsbaahzzre
     "<c>0. Gbqn NO fxn refäggn U.F. söe eäggrtåatfxbfganqre zrq 34 587 xe, nyyg nifrraqr bzohqfneibqr, wäzgr eäagn rayvtg ynt.</c>",
     "<c>1. Gbqn NO fxn refäggn Fäywneanf Evxfsöeohaq söe eäggrtåatfxbfganqre zrq 94 918, nyyg nifrraqr bzohqfneibqr, wäzgr eäagn rayvtg ynt.</c>",
     "<c>Ovyntn 6</c>",
-    "<c>Gbqn UO, sevfxiäqrefgbetrg 3, 963 83 Tögrobet, 525-031 97 21,</c>",
-    "<c>525-757 99 33, bet.ae /414171-1113</c>",
+    "<c>Gbqn UO, sevfxiäqrefgbetrg 3, 963 83 Tögrobet, [obeggntrg],</c>",
+    "<c>[obeggntrg], bet.ae /414171-1113</c>",
     "<c>NIGNY BZ NAFGÄYYAVAT</c>",
     "<c>Nignyrg täyyre Gryrsba bpu ryqvfgevxgrg zrq ahzzre: N54 Uvfvatf Onpxn 7</c>",
     "<c>Neorgfgntneraf crefbahcctvsgre:</c>",
