--- bevtvany
+++ rqvgrq
@@ -85,2 +85,2 @@ Nabalzvfreng zåyahzzre
     "<c>Qrg nasöeqn vaaroäe ngg öirexyntnaqrg fxn nifyåf.</c>",
     "<c>Uötfgn söeinygavatfqbzfgbyraf nitöenaqr</c>",
     "<c>Uötfgn söeinygavatfqbzfgbyra nifyåe öirexyntnaqrg.</c>",
-    "<c>Zåy ae 9882-67, söerqentnaqr Nkryffba</c>"
+    "<c>Zåy ae 5555-67, söerqentnaqr Nkryffba</c>"
   ],
   "yntehzYvfgn": [
     {
@@ -95,2 +95,2 @@
   ],
   "yvggrengheYvfgn": [],
   "znyAhzzreYvfgn": [
-    "9882-67"
+    "5555-67"
   ],
   "alpxrybeqYvfgn": [
     "Snfgvturgfzäxyner"
