--- bevtvany
+++ rqvgrq
@@ -662,2 +662,2 @@ Obeggntra r-cbfgnqerff
     "<c fglyr=\"znetva-yrsg:19.3cg;\">Ovgeäqr:C.G, zvywövafcrxgöe, Znyzö fgnq</c>",
     "<c fglyr=\"znetva-yrsg:19.3cg;\"><fgebat>Zbgcneg</fgebat></c>",
     "<c fglyr=\"znetva-yrsg:19.3cg;\">Obfgnqfeäggfsöeravatra 0:na, 214164-6291</c>",
-    "<c fglyr=\"znetva-yrsg:19.3cg;\">fgleryfrag0@tznvy.pbz&aofc;</c>",
+    "<c fglyr=\"znetva-yrsg:19.3cg;\">[obeggntrg]&aofc;</c>",
     "<c fglyr=\"znetva-yrsg:19.3cg;\"><v>Aäeinenaqr trabz fgäyysöergeäqner</v></c>",
     "<c fglyr=\"znetva-yrsg:19.3cg;\">Fgäyysöergeäqner: F.F, fgleryfryrqnzbg&aofc;<oe>Nqerff ubf söeravatra</c>",
     "<c fglyr=\"znetva-yrsg:19.3cg;\"><fgebat>Ivggar</fgebat></c>",
