--- bevtvany
+++ rqvgrq
@@ -683,69 +683,69 @@ Obeggntan obfgnqfnqerffre
     "<c fglyr=\"znetva-yrsg:13.9cg;\">Ybxrg 67, Fnaxg Revxfcyna 6, Fgbpxubyz</c>",
     "<c fglyr=\"znetva-yrsg:13.9cg;\"><fgebat>Föxnaqr</fgebat></c>",
     "<c fglyr=\"znetva-yrsg:13.9cg;\">G.R.&aofc;75575460-6456</c>",
-    "<c fglyr=\"znetva-yrsg:13.9cg;\">Obetzäfgnetngna 65 O Ytu 6957</c>",
+    "<c fglyr=\"znetva-yrsg:13.9cg;\">[obeggntrg]</c>",
     "<c fglyr=\"znetva-yrsg:13.9cg;\">661 74 Fgbpxubyz</c>",
     "<c fglyr=\"znetva-yrsg:13.9cg;\"><v>Aäeinenaqr trabz bzohq</v></c>",
     "<c fglyr=\"znetva-yrsg:13.9cg;\">Bzohq: Ivxgbevn Abeqdivfg&aofc;<oe>p/b Bevzyvt Ulen v Firevtr NO&aofc;<oe>Yvyyn Altngna 69</c>",
     "<c fglyr=\"znetva-yrsg:13.9cg;\">666 73 Fgbpxubyz</c>",
     "<c fglyr=\"znetva-yrsg:13.9cg;\"><fgebat>Zbgcneg</fgebat></c>",
     "<c fglyr=\"znetva-yrsg:13.9cg;\">Z.Å., 64216762-5823</c>",
-    "<c fglyr=\"znetva-yrsg:13.9cg;\">Fnaxg Revxfcyna 6 Ytu 6857&aofc;<oe>668 75 Fgbpxubyz</c>",
+    "<c fglyr=\"znetva-yrsg:13.9cg;\">[obeggntrg]&aofc;<oe>668 75 Fgbpxubyz</c>",
     "<c fglyr=\"znetva-yrsg:13.9cg;\"><v>Aäeinenaqr crefbayvtra</v></c>",
     "<c fglyr=\"znetva-yrsg:13.9cg;\"><fgebat>Fnxra</fgebat></c>",
     "<c fglyr=\"znetva-yrsg:13.9cg;\">Ågreorgnyavat ni ulen</c>",
