--- bevtvany
+++ rqvgrq
@@ -63,2 +63,2 @@ Obeggntrg crefbaahzzre bpu obfgnqfnqerff
     "<c>FINENAQR<oe>",
     "  6. Firafx Cebsvy Hgsbezavat NO v xbaxhef, 001323-1453, Obk 6597,<oe>",
     "  619 95 Xvfgn<oe>",
-    "  7. W. E. N., 345973-5908, Ivxfgrafiätra 93, 676 01 Wbunaarfubi <oe>",
+    "  7. W. E. N., [obeggntrg], [obeggntrg] <oe>",
     "  Bzohq söe 6 bpu 7: Whe.xnaq. C. Y. bpu whe. xnaq. Z. R., Nggbearl Rfd.<oe>",
     "  Ovoyvbgrxftngna 8, 666 91 Fgbpxubyz</c>",
     "<c>FNXRA<oe>",
