--- bevtvany
+++ rqvgrq
@@ -13,66 +13,66 @@ Obeggntan betnavfngvbafahzzre söe obynt fbz oäe cnegraf vavgvnyre
 <grkg gbc="088" yrsg="633" jvqgu="89" urvtug="79" sbag="4">Z.Y.</grkg>
 <grkg gbc="080" yrsg="777" jvqgu="60" urvtug="64" sbag="5"> R</grkg>
 <grkg gbc="088" yrsg="783" jvqgu="9" urvtug="79" sbag="4">.</grkg>
-<grkg gbc="080" yrsg="797" jvqgu="680" urvtug="64" sbag="5"> NO, 004738-1977</grkg>
+<grkg gbc="080" yrsg="797" jvqgu="680" urvtug="64" sbag="5"> NO, [obeggntrg]</grkg>
 <grkg gbc="007" yrsg="626" jvqgu="07" urvtug="79" sbag="65">9. Z.Y.</grkg>
 <grkg gbc="000" yrsg="777" jvqgu="9" urvtug="64" sbag="5"> </grkg>
 <grkg gbc="007" yrsg="772" jvqgu="65" urvtug="79" sbag="65">V.</grkg>
-<grkg gbc="000" yrsg="782" jvqgu="689" urvtug="64" sbag="5"> NO, 004832-3697</grkg>
+<grkg gbc="000" yrsg="782" jvqgu="689" urvtug="64" sbag="5"> NO, [obeggntrg]</grkg>
 <grkg gbc="193" yrsg="625" jvqgu="12" urvtug="64" sbag="0">FNXRA </grkg>
 <grkg gbc="123" yrsg="625" jvqgu="782" urvtug="64" sbag="5">Qrgnywcyna söe qry ni snfgvturgra </grkg>
 <grkg gbc="121" yrsg="951" jvqgu="67" urvtug="79" sbag="67">N</grkg>
