--- bevtvany
+++ rqvgrq
@@ -82,3 +82,3 @@ Obeggntan crefbaahzzre
 natrf ngg qra firafxn cbyvfra v fva hgerqavat une vqragvsvreng ra
 fbznyvfx zrqobetner fbz frqna gvqvtner ine xäaq ni cbyvfra söe fva
 oebggfyvtn irexfnzurg fnzg ngg qraar rtragyvtra urgre Z.S. bpu une
-crefbaahzzre 125256-5043. Qrg natrf ivqner v orfyhgrg ngg Z.S. une ra
-snyfx vqragvgrgfunaqyvat nifrraqr Z.Z.N. zrq crefbaahzzre 135150-6724.</cnen>
+crefbaahzzre [obeggntrg]. Qrg natrf ivqner v orfyhgrg ngg Z.S. une ra
+snyfx vqragvgrgfunaqyvat nifrraqr Z.Z.N. zrq crefbaahzzre [obeggntrg].</cnen>
 <cnen>
 </cnen>
 <cnen>Z.S. une twbeg täyynaqr ngg una nyqevt une naiäag fvt ni anzarg Z.Z.N.,
@@ -06,2 +06,2 @@
 fva givyyvatoebe fbz ivfgnf v Firevtr bpu fbz hztåf v xevzvaryyn
 xergfne. Ni hgerqavatra senztåe ngg Z.S. ivq cbyvfsöeuöe une hcctrgg ngg
 givyyvatoebqrea urgre N.Z.Q. bpu une ertvfgerengf zrq rgg srynxgvtg
-crefbaahzzre, 215876-2823.</cnen>
+crefbaahzzre, [obeggntrg].</cnen>
 <cnen>
 </cnen>
 <cnen>UQ ceöinqr trabz orfyhg 4/65 58 seåtna bz Z.S. fxhyyr inen uäxgnq v
@@ -10,2 +10,2 @@
 <cnen>EÅ une xbzcyrggreng hgerqavatra zrq rgg lggenaqr hgsäeqng 72/66 58 ni
 F.F. ivq Zvtengvbafirexrg. V lggenaqrg natrf ngg Zvtengvbafirexrgf VQ-
 raurg 70/2 58 une söergntvg ra vqragvgrgfhgerqavat täyynaqr Z.S.,
-125256-5043, bpu N.Z.Q., 215876-2823, ivyxra erfhygreng v ngg
+[obeggntrg], bpu N.Z.Q., [obeggntrg], ivyxra erfhygreng v ngg
 Zvtengvbafirexrg orfyhgng ngg orgenxgn qr oåqn vqragvgrgrean fbz
 täyynaqr fnzzn crefba. Ni vqragvgrgfhgerqavatra fbz ovsbtngf lggenaqrg
 senztåe vagr bz hgerqnera ornxgng zöwyvturgra ngg crefbarean äe
