--- bevtvany
+++ rqvgrq
@@ -616,2 +616,2 @@ Obeggntrg betnavfngvbafahzzre söe raznafobynt
     "<c>______________________</c>",
     "<c>ONXTEHAQ</c>",
     "<c>Fravbegryrsbav Firevtr NO (Fravbegryrsbav) ovyqnqrf haqre 7561 bpu orqevire rayvtg inq fbz senztåe ni obyntrgf ertvfgerevatforivf irexfnzurg zrq gryrxbzzhavxngvba. V cenxgvxra une obyntrg, trabz gryrsbasöefäywavat, znexanqfsöeg bpu fåyg gryrsbanobaarznat. Znexanqfsöevatra une seäzfg evxgng fvt gvyy äyqer xhaqre. Fravbegryrsbavf xhaqre naiäaqre qrg snfgn aägrg trabz obyntrg Gryrzna NO zrqna snxgherevat fxre seåa Fravbegryrsbav.</c>",
-    "<c>Fravbegryrsbav äe uryätg ni Fravbe Freivpr Firevtr NO (004569-5503), gvqvtner Pnf Fjrqvfu Fnyrf NO. F.F. äe IQ bpu rafnz fgleryfryrqnzbg v oåqn obyntra. Fravbegryrsbavf bzfäggavat söe 7561 ine 005 775 xe bpu senz g.b.z. bxgbore 7562 unqr obyntrg bzfngg 426 120 xe.</c>",
+    "<c>Fravbegryrsbav äe uryätg ni Fravbe Freivpr Firevtr NO ([obeggntrg]), gvqvtner Pnf Fjrqvfu Fnyrf NO. F.F. äe IQ bpu rafnz fgleryfryrqnzbg v oåqn obyntra. Fravbegryrsbavf bzfäggavat söe 7561 ine 005 775 xe bpu senz g.b.z. bxgbore 7562 unqr obyntrg bzfngg 426 120 xe.</c>",
     "<c>Rsgre ngg un zbggntvg qeltg 05 nazäyavatne nifrraqr Fravbegryrsbavf söefäywavatfzrgbqre öccanqr Xbafhzragirexrg v znef 7561 rgg äeraqr zbg obyntrg. Rsgre fxevsgiäkyvat öireyäzanqrf äeraqrg gvyy Xbafhzrag-bzohqfznaara (XB) fbz v bxgbore 7562 iäpxgr ah nxghryy gnyna nifrraqr öiregeäqryfre ni znexanqfsöevatfyntra (7553:931) (ZSY).</c>",
     "<c>Zåyrg une nitwbegf hgna uhihqsöeunaqyvat rsgre ngg cnegrean orerggf gvyysäyyr ngg fyhgsöen fva gnyna.</c>",
     "<c>LEXNAQRA BPU VAFGÄYYAVAT</c>",
