--- bevtvany
+++ rqvgrq
@@ -93,4 +93,4 @@ Obeggntan sbyxobxsöevatfnqerffre
 rgg fåqnag sry haqre zåyrgf unaqyättavat fbz hgtöe tehaq söe erfavat. -
 Fxnggrirexrg gvyyfglexgr ovsnyy gvyy lexnaqrg bz ågresöeivfavat zra
 nafåt ngg öirexyntnaqrg v öievtg fxhyyr nifyåf. Irexrg nasöeqr oy.n.
-söywnaqr. N.Y. ine sbyxobxsöeq cå nqerffra Öier Uhfnetngna 75 g.b.z. qra
+söywnaqr. N.Y. ine sbyxobxsöeq cå nqerffra Öier [obeggntrg] g.b.z. qra
 78 znef 7551. Seåa aäzaqn qnghz äe N.Y. sbyxobxsöeq cå nqerffra
-Pyrzragvatngna 70. Qra haqreeäggryfr, qngrenq qra 75 qrprzore 7551, fbz
+[obeggntrg]. Qra haqreeäggryfr, qngrenq qra 75 qrprzore 7551, fbz
 xnzzneeäggra fxvpxnqr gvyy N.Y. fxvpxnqrf gvyy nqerffra cå Öier
 Uhfnetngna. N.Y. unqr vagr haqreeäggng xnzzneeäggra bz nqerffäaqevatra
 bpu söefäaqryfra xbz vagr v erghe. - Qrg senztåe vagr ni söeneorgra inq
