--- bevtvany
+++ rqvgrq
@@ -73,3 +73,3 @@ Obeggntan obfgnqfnqerffre
 Aåtba ceöiavat ni zbgfgåraqr nyyzäaan vagerffra söehgfäggf vagr fxr v
 ceöiavatra ni rgg orfiäefäeraqr. - Ni hgerqavatra senztåe ngg xyntnaqran
 vagr äe hccgntan v snfgvturgfsöegrpxavatra. Ivqner senztåe ngg
-Fvtheq Q:f obfgnqfnqerff äe Fxlggrubyzfiätra 3, ivyxra äe oryätra vabz
-begra Aäefgn cå Rxreö, bpu ngg Cre R:f obfgnqfnqerff äe Sybeniätra 64,
+Fvtheq Q:f obfgnqfnqerff äe [obeggntrg], ivyxra äe oryätra vabz
+begra Aäefgn cå Rxreö, bpu ngg Cre R:f obfgnqfnqerff äe [obeggntrg],
 ivyxra äe oryätra vabz begra Gnccfgeöz. Xbzzhara une qbpx rsgre
 nagntnaqrorfyhgrg fxvpxng haqreeäggryfr bz orfyhgrg zrq
 shyysöywqfuäaivfavat gvyy oy.n. xyntnaqran. - Fvtheq Q. äe obfngg zre äa
