--- data/sfs/intermediate/1944/181.txt	2009-09-26 23:53:30.519000000 +0200
+++ c:\users\staffan\appdata\local\temp\tmpwjtiw4	2009-09-26 23:52:55.099300000 +0200
@@ -14,7 +14,7 @@
 Vad den som mottagit medel för annan, med skyldighet att redovisa för
 dem, för sådant ändamål har innestående på räkning i bank eller
 annorledes håller avskilt skall vara förbehållet huvudmannen, såvitt
-beloppet avskiltt utan dröjsmål. Samma lag vare i fråga om belopp som
+beloppet avskilts utan dröjsmål. Samma lag vare i fråga om belopp som
 avskilts senare, om den redovisningsskyldige ej var på obestånd, när
 beloppet avskildes.
 
