--- data/sfs/intermediate/1960/729.txt
+++ /tmp/tmp557g3hr9
@@ -641,7 +641,7 @@ Korrigerat stavfel i 26 a § 4 p
 
 3. datorprogram,
 
-4. öerk som skapats för undervisning,
+4. verk som skapats för undervisning,
 
 5. verk som är resultatet av vetenskaplig forskning,
 
@@ -1244,6 +1244,7 @@
 upphovsmannen motsätter sig förfogandet. Lag (2013:691).
 
 4 Kap. Upphovsrättens giltighet
+
 43 § Upphovsrätt till ett verk gäller till utgången av
 sjuttionde året efter det år då upphovsmannen avled eller, i
 fråga om verk som avses i 6 §, efter den sist avlidne
