--- data/sfs/intermediate/1960/729.txt	2016-04-04 21:57:36.621998543 +0200
+++ /tmp/tmpvmzgmR	2016-04-04 21:57:30.554286000 +0200
@@ -633,7 +633,7 @@
 
 3. datorprogram,
 
-4. öerk som skapats för undervisning,
+4. verk som skapats för undervisning,
 
 5. verk som är resultatet av vetenskaplig forskning,
 
@@ -1236,6 +1236,7 @@
 upphovsmannen motsätter sig förfogandet. Lag (2013:691).
 
 4 Kap. Upphovsrättens giltighet
+
 43 § Upphovsrätt till ett verk gäller till utgången av
 sjuttionde året efter det år då upphovsmannen avled eller, i
 fråga om verk som avses i 6 §, efter den sist avlidne
