--- data/sfs/intermediate/1960/729.txt
+++ /tmp/tmp557g3hr9
@@ -1244,6 +1244,7 @@ [version: 2014:884-2018:604] Formattering, infogad nyrad innan 43 §
 upphovsmannen motsätter sig förfogandet. Lag (2013:691).
 
 4 Kap. Upphovsrättens giltighet
+
 43 § Upphovsrätt till ett verk gäller till utgången av
 sjuttionde året efter det år då upphovsmannen avled eller, i
 fråga om verk som avses i 6 §, efter den sist avlidne
