--- data/sfs/intermediate/1969/644.txt
+++ /tmp/tmpn5xey6kh
@@ -36,7 +36,7 @@ [version: 1996:1632-1996:1632] Korrigerat SFS-nummer i hänvisning till rättshjälpslagen
 Förordnande enligt första stycket första punkten får avse endast den
 som stadigvarande vistas i riket, om ej särskilda skäl föranleder
 annat. Vad som sagts nu gäller ej i fråga om rättshjälp enligt
-rättshjälpslagen (1996:000) och ej heller i fråga om annan rättighet,
+rättshjälpslagen (1996:1619) och ej heller i fråga om annan rättighet,
 om den hänför sig till rättegång vid svensk domstol. Lag (1996:1632).
 
 
