--- data/sfs/intermediate/1969/644.txt	2009-11-11 17:14:02.144000000 +0100
+++ c:\users\staffan\appdata\local\temp\tmpj0cdnw	2009-11-11 17:13:57.058899900 +0100
@@ -28,5 +28,5 @@
 Förordnande enligt första stycket första punkten får avse endast den
 som stadigvarande vistas i riket, om ej särskilda skäl föranleder
 annat. Vad som sagts nu gäller ej i fråga om rättshjälp enligt
-rättshjälpslagen (1996:000) och ej heller i fråga om annan rättighet,
-om den hänför sig till rättegång vid svensk domstol. Lag (1996:1632).
+rättshjälpslagen (1996:1619) och ej heller i fråga om annan rättighet,
+om den hänför sig till rättegång vid svensk domstol. Lag (1996:1632).
