--- data/sfs/intermediate/1981/774.txt.bz2
+++ 
@@ -749,6 +749,7 @@ Korrigerat styckeindelning i 4:2
 
 2 § Genom utmätning får tagas i anspråk egendom som ej är undantagen på
 grund av bestämmelse i denna balk eller på grund av särskild föreskrift.
+
 I den mån pantbrev i skepp eller fastighet ej utgör säkerhet för fordran
 får det utmätas hos den intecknade egendomens ägare. Motsvarande gäller
 skuldebrev som är intecknat i luftfartyg eller reservdelar till
