--- data/sfs/intermediate/1993/1633.txt	2009-11-11 17:18:59.511000000 +0100
+++ c:\users\staffan\appdata\local\temp\tmpjvp5ew	2009-11-11 17:18:35.020099900 +0100
@@ -52,11 +52,11 @@
 första stycket.
 
 7 § Om polismyndigheten enligt 2 kap. 12 § fjärde stycket
-ordningslagen (1993:000) förbjuder att ett tält används, skall
+ordningslagen (1993:1617) förbjuder att ett tält används, skall
 myndigheten infordra godkännandebeviset.
 
 Ett beslut om användningsförbud gäller genast.
 
 8 § Närmare föreskrifter om besiktning av samlingstält meddelas
 av Myndigheten för samhällsskydd och beredskap efter samråd med
-Rikspolisstyrelsen och Boverket. Förordning (2008:1024).
+Rikspolisstyrelsen och Boverket. Förordning (2008:1024).
