--- data/sfs/intermediate/1979/1152.txt.bz2
+++ /tmp/tmpxa6hhgrp
@@ -222,13 +222,15 @@ Justering
 		annan byggnad för produktionen utgör 
 		kraftverksbyggnad.	
 
-Industribyggnad	Byggnad som är inrättad för industriell 
+Industri-
+byggnad		Byggnad som är inrättad för industriell 
 		verksamhet och som inte utgör 
 		kraftverksbyggnad.
 
 Specialbyggnad	Med specialbyggnad avses		
 
-		Försvarsbyggnad	Byggnad som används för 
+		Försvars-
+		byggnad		Byggnad som används för 
 				försvarsändamål eller 
 				ekonomisk försvarsberedskap, 
 				om byggnaden inte är en 
