--- data/sfs/intermediate/1970/215.txt	2009-10-08 14:14:18.305000000 +0200
+++ c:\users\staffan\appdata\local\temp\tmpapmt7l	2009-10-08 14:13:47.007599900 +0200
@@ -24,7 +24,7 @@
 
 2 § Kvittning mot lönefordran får ske med arbetstagarens medgivande.
 Återkallas ett på förhand lämnat medgivande innan lönefordringen
-förfallit till betalning, får kvitning icke ske i vidare mån än som
+förfallit till betalning, får kvittning icke ske i vidare mån än som
 följer av vad nedan sägs.
 
 3 § Kan kvittning ej ske enligt 2 §, får arbetsgivaren kvitta endast med
@@ -39,8 +39,8 @@
 Rätt till kvittning utöver vad som följer av första och andra styckena
 kan medges arbetsgivare genom kollektivavtal som på arbetstagarsidan
 slutits eller godkänts av organisation som enligt lagen (1976:580) om
-medbestämmande i arbetslivet är att anse som central arbetstagarorga
-nisation. Arbetsgivare som är bunden av sådant kollektivavtal får
+medbestämmande i arbetslivet är att anse som central arbetstagarorganisation.
+Arbetsgivare som är bunden av sådant kollektivavtal får
 tillämpa det även i fråga om arbetstagare som ej är medlem av den
 avtalsslutande arbetstagarorganisationen men sysselsättes i arbete som
 avses med avtalet. Detta gäller dock ej arbetstagare som omfattas av
@@ -51,7 +51,7 @@
 vad som åtgår för arbetstagarens och hans familjs försörjning samt till
 fullgörande av underhållsskyldighet som i övrigt åvilar honom.
 
-Regeringen eller den myndighet regeringen bestämmer meddelar när  mare
+Regeringen eller den myndighet regeringen bestämmer meddelar närmare
 föreskrifter om beräkningen av hur stor del av lönefordringen som skall
 vara skyddad mot kvittning. Lag (1976:92).
 
