--- data/sfs/intermediate/1988/97.txt	2009-11-11 17:16:16.105000000 +0100
+++ c:\users\staffan\appdata\local\temp\tmplqndyv	2009-11-11 17:15:58.046499900 +0100
@@ -289,7 +289,7 @@
 som inte annan sålunda förpliktas betala, skall återbetalas till
 staten av den för vilken ombudet har förordnats, dock inte mer än vad
 han skulle ha fått betala i rättshjälpsavgift enligt rättshjälpslagen
-(1996:000). Vad som där sägs om rättshjälpsbiträde gäller kostnad för
+(1996:1619). Vad som där sägs om rättshjälpsbiträde gäller kostnad för
 ombudet.
 
 Om ersättningen till ombudet understiger ett visst av regeringen
