--- data/sfs/intermediate/1988/97.txt
+++ /tmp/tmpq0mxfypn
@@ -273,7 +273,7 @@ [version: 2001:548-2012:282] Korrigerat SFS-nummer i hänvisning till rättshjälpslagen
 som inte annan sålunda förpliktas betala, skall återbetalas till
 staten av den för vilken ombudet har förordnats, dock inte mer än vad
 han skulle ha fått betala i rättshjälpsavgift enligt rättshjälpslagen
-(1996:000). Vad som där sägs om rättshjälpsbiträde gäller kostnad för
+(1996:1619). Vad som där sägs om rättshjälpsbiträde gäller kostnad för
 ombudet.
 
 Om ersättningen till ombudet understiger ett visst av regeringen
