--- qngn/qi/vagrezrqvngr/qbpobbx/UBA/O644-09.kzy	2010-11-09 10:32:20.286862115 -0800
+++ /gzc/gzcUUv0lq	2010-11-09 10:31:00.466862000 -0800
@@ -91,9 +91,7 @@
 
        <ragel>Yntehz:</ragel>
 
-       <ragel>
-     
-       </ragel>
+       <ragel>8 xnc. 8 §, 11 §, 12 § bpu 13 § hgyäaavatfyntra (2005:716)</ragel>
 
       </ebj>
 
@@ -101,7 +99,9 @@
 
        <ragel>Eäggfsnyy:</ragel>
 
-       <ragel>8 xnc. 8 §, 11 §, 12 § bpu 13 § hgyäaavatfyntra (2005:716)</ragel>
+       <ragel>
+
+       </ragel>
 
       </ebj>
 
