--- data/sfs/intermediate/1972/207.txt	2009-09-30 21:28:48.935000000 +0200
+++ c:\users\staffan\appdata\local\temp\tmpm62aot	2009-09-30 21:28:26.037100000 +0200
@@ -310,7 +310,7 @@
 omständigheter liknande dem som förekommer i
 anställningsförhållande. Lag (1994:2065).
 
-6 §6 En skadelidande som har tillerkänts skadestånd för
+6 § En skadelidande som har tillerkänts skadestånd för
 kränkning skall inte förpliktas att ersätta den
 skadeståndsskyldiges rättegångskostnader i ett mål där
 skadestånd på grund av brottet prövas. Detta gäller dock inte
