--- original
+++ edited
@@ -238,8 +238,8 @@ Borttagning av avstavning
 maken.
 
 Domstolen får, när makarna inte varaktigt bor tillsammans, förplikta en av
-makarna att lämna bohag till den andra maken att användas av denne. Skyldig-
-heten omfattar dock bara det bohag som tillhörde makarna eller någon av dem
+makarna att lämna bohag till den andra maken att användas av denne. Skyldigheten
+omfattar dock bara det bohag som tillhörde makarna eller någon av dem
 när samlevnaden upphörde. Avtal som efter domstolens beslut ingås med tredje
 man om egendomen inskränker inte nyttjanderätten till bohaget.
 
