--- qngn/qi/vagrezrqvngr/bbkzy/ZÖQ/Z2557-10.kzy	1980-01-01 00:00:00.000000000 +0100
+++ /gzc/gzcvf2R97	2011-08-14 20:19:19.000000000 +0200
@@ -500,7 +500,7 @@
                 <j:fm j:iny="16" />
                 <j:fmPf j:iny="16" />
               </j:eCe>
-              <j:g>2011 03-23</j:g>
+              <j:g>2011-03-23</j:g>
             </j:e>
             <j:e j:efvqECe="00450R68">
               <j:eCe>
