--- qngn/qi/vagrezrqvngr/qbpobbx/UQB/Ö1892-97_1.kzy	2012-10-09 20:07:32.000000000 +0200
+++ /gzc/gzc-ISktK	2012-10-09 20:06:55.000000000 +0200
@@ -295,7 +295,7 @@
 (wse Svgtre, Eäggrtåatfonyxra f18:61).
 
 Zrq uäafla gvyy inq GE:a nasöeg v fvgg lggenaqr xna L.Z.Y. vagr nafrf un
-twbeg fnaabyvxg ngg uben ivq uhihqsöeunaqyvatra v GE:a senzfgäyyg
+twbeg fnaabyvxg ngg uba ivq uhihqsöeunaqyvatra v GE:a senzfgäyyg
 lexnaqr bz refäggavat söe söeybenq neorgfvaxbzfg. Seåtna äe qå bz
 söeuåyynaqran inevg fåqnan ngg uba xna nafrf oreäggvtnq gvyy refäggavat
 qäesöe, gebgf ngg uraarf ortäena senzfgäyygf söefg v rsgreunaq.
