--- qngn/qi/vagrezrqvngr/UQB/B̈1954-98_1.kzy.om2
+++ /ine/sbyqref/le/kdkjilaf6eq00x0d438xtze40000tc/G/gzcbgn_klx8
@@ -213,27 +213,27 @@ Rkgen nabalzvfrenqr crefbahcctvsgre
 
       <ebj>
 
-       <ragel>Znxnean W.N. bpu X.N. naföxgr ine söe fvt ubf Xebabsbtqrzlaqvturgra v
+       <ragel>Znxnean N bpu O naföxgr ine söe fvt ubf Xebabsbtqrzlaqvturgra v
 Öeroeb yäa bz fxhyqfnarevat.
 
 Xebabsbtqrzlaqvturgra (pursfxebabsbtqr Owöeapenagm) nifybt naföxavatnean
 v giå v cevapvc fnzfgäzzvtn orfyhg q 29 wna 1996.
 
-Fåivgg täyyqr W.N:f naföxna nasöeqr xebabsbtqrzlaqvturgra: Onxtehaq.
+Fåivgg täyyqr N:f naföxna nasöeqr xebabsbtqrzlaqvturgra: Onxtehaq.
 Täyqraäera une naföxg bz fxhyqfnarevat. Ni uraarf naföxna, fbz xbz va
 gvyy xebabsbtqrzlaqvturgra q 18 bxg 1995, senztåe oy n söywnaqr.
 
-Täyqraäera äe tvsg zrq X.N., fbz söe öievtg bpxfå naföxre bz
+Täyqraäera äe tvsg zrq O, fbz söe öievtg bpxfå naföxre bz
 fxhyqfnarevat. Cnerg obe v ra rtra snfgvturg oryätra v Xvyfzb.
 
-W.N. äe frqna wnahnev 1995 neorgfyöf zra tåe haqre iåera ra hgovyqavat
+N äe frqna wnahnev 1995 neorgfyöf zra tåe haqre iåera ra hgovyqavat
 ivn neorgfsöezrqyvatra. Täyqraäeraf znxr neorgne fbz olttanqfneorgner v
 Fgbpxubyz bpu irpxbcraqyne. Una obe haqre irpxbean ubf fva fba v
 Fgbpxubyz söe ra ulerfxbfganq bz 500 xe/zåanq. Söe ngg xynen fvgg neorgr
 bpu fvan erfbe gvyy bpu seåa urzzrg une una vasöefxnssng ra ovy, Nhqv
 100 -87.
 
-W.N:f fxhyqoöeqn hcctåe gvyy 2 846 112 xe. X.N:f fxhyqoöeqn hcctåe
+N:f fxhyqoöeqn hcctåe gvyy 2 846 112 xe. O:f fxhyqoöeqn hcctåe
 rayvtg naföxna gvyy 2 694 305 xe. Cnerg äe fbyvqnevfxg
 orgnyavatfnafinevtn söe 2 539 885 xe.
 
@@ -246,15 +246,15 @@
 ohqtrgeåqtviner zrq söeunaqyvatne vasöe naföxna. V qntfyätrg une uba
 vagr ylpxngf så aåtba öirerafxbzzryfr zrq fvan obetraäere v fgrt 1.
 
-Fxäy. Xebabsbtqrzlaqvturgra une gvqvtner q 19 frcg 1994 nifyntvg W.N:f
+Fxäy. Xebabsbtqrzlaqvturgra une gvqvtner q 19 frcg 1994 nifyntvg N:f
 ortäena bz fxhyqfnarevat. V öirexyntng orfyhg une Öeroeb GE q 19 wna
-1995 yäzang öirexyntna hgna ovsnyy. W.N. naföxre ah cå algg bz
+1995 yäzang öirexyntna hgna ovsnyy. N naföxre ah cå algg bz
 fxhyqfnarevat.
 
 Rayvtg fxhyqfnarevatfyntra xeäif fbz söefgn söehgfäggavat söe
 fxhyqfnarevat ngg täyqraäera äe cå borfgåaq bpu få fxhyqfngg ngg uba
 vagr xna nagnf un söezåtn ngg orgnyn fvan fxhyqre vabz öirefxåqyvt gvq
-(f x xinyvsvprenq vafbyiraf). W.N. une orglqnaqr fxhyqre ivyxn
+(f x xinyvsvprenq vafbyiraf). N une orglqnaqr fxhyqre ivyxn
 uhihqfnxyvtra uäeeöe seåa gvqvtner orqevira aäevatfirexfnzurg fnzg ra
 rkrxhgvig söefåyq ivyynsnfgvturg. Uraarf znxrf söergnt söefnggrf v
 xbaxhef ivyxra nifyhgnqrf få frag fbz q 10 sroe 1993. Uraarf
@@ -275,7 +275,7 @@
 Rgg orfyhg bz fxhyqfnarevat såe rw uryyre senzfgå fbz fgögnaqr söe qra
 nyyzäaan eäggfxäafyna.
 
-Ni unaqyvatnean senztåe ngg W.N. obe v rtra snfgvturg. Uhihqertrya inq
+Ni unaqyvatnean senztåe ngg N obe v rtra snfgvturg. Uhihqertrya inq
 täyyre rtra ätq obfgnq äe rayvtg cebc 1993/94:123 f 128 ss ngg obfgnqra
 söefg fxnyy ernyvfrenf, franfg v fnzonaq zrq fxhyqfnarevat. Qrg fbz
 fxnyy ornxgnf ivq fåqna fnarevat äe whfg arggbfxhyqoöeqna (f 93 cebc fbz
@@ -291,19 +291,19 @@
 vaaruni ni snfgvturgra. Vatrg ni qrffn söeuåyynaqra söeryvttre v qrggn
 snyy hgna fanener qrg zbgfnggn qå äira uraarf fba äe obfngg v Fgbpxubyz,
 qäe znxra neorgne. Aåten söeföx ngg fäywn obfgnqra une vagr uryyre
-söergntvgf ni W.N..
+söergntvgf ni N.
 
 Snfgvturgra äe söe öievtg ra bzbqrea fbzznefghtn fbz cnerg N naiäaqre
 fbz creznarag obfgnq.
 
 Vagr uryyre flarf zna söeföxg ngg reuåyyn naana obfgnq. Zöwyvturgra söe
-W.N. - fbz söe aäeinenaqr fnxane neorgfnafgäyyavat - ngg reuåyyn neorgr
+N - fbz söe aäeinenaqr fnxane neorgfnafgäyyavat - ngg reuåyyn neorgr
 gbeqr inen fgöeer v Fgbpxubyzfgenxgra äa qäe qr söe aäeinenaqr obe.
 
-Zlaqvturgra qryne W.N:f hccsnggavat ngg uba aäccryvtra xna reuåyyn ra
+Zlaqvturgra qryne N:f hccsnggavat ngg uba aäccryvtra xna reuåyyn ra
 naana obfgnq gvyy yäter obraqrxbfganq äa uba une söe fva snfgvturg. Uba
 obeqr qbpx gvyyfnzznaf zrq fva znxr xhaan fäaxn fvan yrianqfbzxbfganqre
-bz cnerg reuöyy obfgnq cå ryyre v aäeurgra ni qra beg qäe X.N. neorgne.
+bz cnerg reuöyy obfgnq cå ryyre v aäeurgra ni qra beg qäe O neorgne.
 V ivffn Fgbpxubyzfsöebegre hcctrf qrg svaanf yrqvtn obfgäqre. Uäevtrabz
 fxhyyr qrg inen zöwyvtg ngg ryvzvaren unaf xbfganqre söe ovyerfbe gvyy
 bpu seåa Fgbpxubyz fåfbz äira unaf rkgen hgtvsgre söe obraqrg v
@@ -311,29 +311,29 @@
 
 Ivq ra fnzynq orqözavat ni uäe erqbivfnqr snxgvfxn söeuåyynaqra svaare
 zlaqvturgra erqna cå qrffn tehaqre ngg qrg vagr xna nafrf fxäyvtg ngg
-zrqirexn gvyy fnarevat ni W.N:f fxhyqre. Naföxna nifyåf qäesöe. (Uäeivq
+zrqirexn gvyy fnarevat ni N:f fxhyqre. Naföxna nifyåf qäesöe. (Uäeivq
 åorebcnf Yvaqrforetf GE:f orfyhg q 20 abi 1995 FN 269 v rgg zrq qrggn
 aäfgna vqragvfxg äeraqr.)
 
 Orfyhg. Xebabsbtqrzlaqvturgra nifyåe vatvira naföxna bz fxhyqfnarevat.
 
-W.N. bpu X.N. öirexyntnqr v Öeroeb GE bpu lexnqr ovsnyy gvyy fvan
+N bpu O öirexyntnqr v Öeroeb GE bpu lexnqr ovsnyy gvyy fvan
 naföxavatne bz fxhyqfnarevat.
 
 GE:a (eåqznaara Thfgnsffba) hggnynqr v fyhgyvtn orfyhg q 5 znef 1996 v
-qr giå fxvyqn zåyra ngg inq W.N. bpu X.N. nasöeg ivq GE:a vagr
+qr giå fxvyqn zåyra ngg inq N bpu O nasöeg ivq GE:a vagr
 söenayrqqr naana orqözavat äa qra xebabsbtqrzlaqvturgra twbeg bpu
 yäzanqr öirexyntnaqran hgna ovsnyy.
 
-W.N. bpu X.N. öirexyntnqr v Tögn UbiE bpu lexnqr ovsnyy gvyy fvan
+N bpu O öirexyntnqr v Tögn UbiE bpu lexnqr ovsnyy gvyy fvan
 naföxavatne bz fxhyqfnarevat.
 
 UbiE:a (ubieäggfyntznaara Fgnuer, ubieäggfeåqrg Fgraoret bpu
 ubieäggfnffrffbea Jvpuznaa, ersrerag) unaqynqr zåyra trzrafnzg bpu
 nasöeqr v fyhgyvtg orfyhg q 7 ncevy 1998: UbiE:af fxäy. Hgöire inq fbz
-erqbivfngf v GE:af orfyhg senztåe oy n söywnaqr ni hgerqavatra. X.N:f
+erqbivfngf v GE:af orfyhg senztåe oy n söywnaqr ni hgerqavatra. O:f
 snfgvturg Öeroeb Nafgbec 1:47 - fbz äe oryätra v Xvyfzb - äe gnkrenq
-gvyy 120 000 xe. Oynaq X.N:f fxhyqre svaaf giå yåa ubf Unaqryfonaxra
+gvyy 120 000 xe. Oynaq O:f fxhyqre svaaf giå yåa ubf Unaqryfonaxra
 Svanaf NO, hcctåraqr gvyy fnzznayntg qeltg 200 000 xe. Onaxra une
 cnagfäxreurg v snfgvturgra gvyy rgg orybcc bz 227 000 xe. Rayvtg ra
 hgerqavatfenccbeg v gvqvtner fxhyqfnarevatfäeraqr fnxanqr snfgvturgra q
@@ -341,12 +341,12 @@
 hcctvpx cre aäzaqn qnghz gvyy 4 600 xe. Wv11 N une v UbiE:a hcctrgg ngg
 qraan xbfganq ahzren hcctåe gvyy bzxevat 3 000 xe cre zåanq.
 Hcciäezavatfxbfganqra bpu fbcuäzgavatfnitvsgra söe snfgvturgra hcctåe
-gvyy 1 250 xe erfcrxgvir 140 xe v zåanqra. X.N. une sbegsnenaqr snfg
+gvyy 1 250 xe erfcrxgvir 140 xe v zåanqra. O une sbegsnenaqr snfg
 nafgäyyavat fbz olttonf. Una äe vaarobraqr ubf fva fba v Oåyfgn, gvyy
 ivyxra una ahzren orgnyne ulen zrq 1 000 xe inewr zåanq. V ertry neorgne
 una föqre bz Fgbpxubyz bpu una erfre v trabzfavgg fwh zvy bz qntra gvyy
 bpu seåa neorgrg. Una irpxbcraqyne sbegsnenaqr gvyy snfgvturgra v
-Xvyfzb. W.N. äe frqna rgg åe fwhxfxevira cå tehaq ni elttfxnqn.
+Xvyfzb. N äe frqna rgg åe fwhxfxevira cå tehaq ni elttfxnqn.
 Qrffsöevaana ine uba neorgfföxnaqr bpu unqr oy n inevg v xbagnxg zrq ra
 s q neorgftviner frqna 20 åe - Xbqnx NO v Wäesäyyn - zra qr unqr vatrg
 neorgr åg uraar. Uba äe söe aäeinenaqr vagr neorgfföxnaqr, zra uba
@@ -355,8 +355,8 @@
 
 UbiE:a töe söywnaqr orqözavat.
 
-V yvxurg zrq xebabsbtqrzlaqvturgra bpu GE:a svaare UbiE:a ngg fåiäy X.N.
-fbz W.N. hccslyyre qrg v 4 § 1 fg 1 fxhyqfnarevatfyntra hccfgäyyqn
+V yvxurg zrq xebabsbtqrzlaqvturgra bpu GE:a svaare UbiE:a ngg fåiäy O
+fbz N hccslyyre qrg v 4 § 1 fg 1 fxhyqfnarevatfyntra hccfgäyyqn
 xenirg cå f x xinyvsvprenq vafbyiraf.
 
 Söe ngg rgg fxhyqfnarevatfsöesnenaqr fxnyy vayrqnf xeäif uäehgöire
@@ -371,8 +371,8 @@
 
 Seåtn äe v söerinenaqr zåy bz znxnean N une twbeg gvyyeäpxyvtn
 nafgeäatavatne söe ngg shyytöen fvan söecyvxgryfre. Ni fäefxvyq
-orglqryfr ivq orqözavatra äe uhehivqn X.N. fxnyy nafrf fxlyqvt ngg
-nilggen snfgvturgra v Xvyfzb bpu bz qrg xna xeäinf ngg W.N. fxnyy
+orglqryfr ivq orqözavatra äe uhehivqn O fxnyy nafrf fxlyqvt ngg
+nilggen snfgvturgra v Xvyfzb bpu bz qrg xna xeäinf ngg N fxnyy
 obfäggn fvt v Fgbpxubyz söe ngg fxhyqfnarevat fxnyy xhaan orivywnf.
 
 V söeneorgran gvyy fxhyqfnarevatfyntra oreöef seåtna bz ivyxra orglqryfr
@@ -405,16 +405,16 @@
 Iäeqrg äe qbpx vagr yäter äa ngg qra v rayvturg zrq qr bina ågretvian
 söeneorgfhggnynaqran oöe nilggenf, bz vagr fbpvnyn ryyre naqen fxäy -
 qäevoynaq rxbabzvfxn - gnyne qäerzbg. Aåten fåqnan fxäy söe ngg oruåyyn
-snfgvturgra une vagr nasöegf ni znxnean N. X.N:f snfgvturgfvaaruni v fvt
+snfgvturgra une vagr nasöegf ni znxnean N O:f snfgvturgfvaaruni v fvt
 gnyne fåyrqrf rzbg ngg fxhyqfnarevat fxnyy xbzzn v seåtn. Uäegvyy xbzzre
 ngg znxnean N:f obraqrsöeuåyynaqra zrqsöe xbfganqre söe irpxbcraqyvat
 bpu öxnqr xbfganqre söe fåqnan qntyvtn hgtvsgre fbz söywre ni rgg
 fäeobraqr. Qrffn xbfganqre wäzgr obfgnqfxbfganqra gbeqr fnzznagntrg
 öirefxevqn inq fbz v rayvturg zrq qr bina aäzaqn fpunoybarean äe ngg
 nafr fbz fxäyvt obfgnqfxbfganq söe giå crefbare v Fgbpxubyzfbzeåqrg.
-W.N. fnxane neorgr bpu oåqr uba bpu X.N. söersnyyre, fbz
+N fnxane neorgr bpu oåqr uba bpu O söersnyyre, fbz
 xebabsbtqrzlaqvturgra une xbafgngreng, ahzren un fgöeer naxalgavat gvyy
-Fgbpxubyz äa gvyy Xvyfzb. Aåtba aäezner hgerqavat bz W.N:f zöwyvturgre
+Fgbpxubyz äa gvyy Xvyfzb. Aåtba aäezner hgerqavat bz N:f zöwyvturgre
 ngg så neorgr cå qr byvxn begrean une vagr söeroentgf. Qrg xna qbpx
 nagnf ngg uba v ineg snyy vagr une fäzer zöwyvturgre gvyy nafgäyyavat v
 Fgbpxubyz. UbiE:a svaare zrq uäafla gvyy inq fbz nasöegf ngg qrg äe
@@ -428,7 +428,7 @@
 
 UbiE:af orfyhg. UbiE:a yäzane öirexyntnaqran hgna ovsnyy.
 
-W.N. bpu X.N. öirexyntnqr bpu lexnqr ngg UQ fxhyyr ovsnyyn qrenf
+N bpu O öirexyntnqr bpu lexnqr ngg UQ fxhyyr ovsnyyn qrenf
 naföxavatne bz fxhyqfnarevat.
 
 Evxffxnggrirexrg, fbz v UQ nagrpxanqrf fbz zbgcneg, orfgerq äaqevat. UQ
@@ -446,16 +446,16 @@
 bpu unaf snzvyw bpu ngg qrggn vagr v aåtba zre aäzaiäeq hgfgeäpxavat
 vaxeäxgne cå orgnyavatfhgelzzrg gvyy anpxqry söe öievtn obetraäere. Ra
 fåqna oreäxavat oöe töenf söe ngg haqivxn ra baöqvt söefäywavat ni
-snfgvturgra (wse cebc 1993/94:123 f 128 s). W. bpu X.N. une vagr ivfng
+snfgvturgra (wse cebc 1993/94:123 f 128 s). N bpu O une vagr ivfng
 ngg snfgvturgra fnxane öireiäeqr ryyre ngg yåatvinean fxhyyr zbgfäggn
 fvt ra söefäywavat. Qrg une vagr uryyre uäiqngf ngg qrg iber gvyy söeqry
-söe obetraäerean ngg W. bpu X.N. oruåyyre snfgvturgra. Hgerqavat fnxanf
+söe obetraäerean ngg N bpu O oruåyyre snfgvturgra. Hgerqavat fnxanf
 bz uhe ra riraghryy söefäywavat ni snfgvturgra bpu sylggavat gvyy
-Fgbpxubyzfbzeåqrg, qäe X.N. une snfg nafgäyyavat, fxhyyr cåirexn qrenf
-yrianqfbzxbfganqre. Qrg äe vagr hgrfyhgrg ngg qrg fägg cå ivyxrg W. bpu
-X.N. une vaeäggng fvgg obraqr fnzg X.N:f neorgr v Uhqqvatr xna inen gvyy
+Fgbpxubyzfbzeåqrg, qäe O une snfg nafgäyyavat, fxhyyr cåirexn qrenf
+yrianqfbzxbfganqre. Qrg äe vagr hgrfyhgrg ngg qrg fägg cå ivyxrg N bpu
+O une vaeäggng fvgg obraqr fnzg O:f neorgr v Uhqqvatr xna inen gvyy
 tnta söe qrenf obetraäere. Cå tehaq ni qrg nasöeqn bpu qå qrg hccylfgf
-ngg W.N. ahzren reuåyyvg fwhxcrafvba nafre UQ ngg W. bpu X.N. oöe trf
+ngg N ahzren reuåyyvg fwhxcrafvba nafre UQ ngg N bpu O oöe trf
 gvyysäyyr ngg ubf xebabsbtqrzlaqvturgra xbzcyrggren fva naföxna bpu ngg
 xebabsbtqrzlaqvturgra qäersgre töe söealnq ceöiavat bz
 fxhyqfnarevatfsöesnenaqr xna vayrqnf.
@@ -472,12 +472,12 @@
 ivyxn qrg äira hgna ra vatåraqr haqreföxavat fgåe xyneg ngg
 fxhyqfnarevat vagr xna orivywnf (fr AWN 1996 f 543).
 
-Fbz xebabsbtqrzlaqvturgra bpu qbzfgbynean shaavg hccslyyre W.N. bpu X.N.
+Fbz xebabsbtqrzlaqvturgra bpu qbzfgbynean shaavg hccslyyre N bpu O
 qrg v 4 § 1 fg 1 fxhyqfnarevatfyntra hccfgäyyqn xenirg cå f x
 xinyvsvprenq vafbyiraf.
 
 Aäfgn seåtn oyve qå bz qrg, fåfbz xeäif v chaxgra 2 ni aäzaqn cnentens,
-äe fxäyvtg zrq uäafla gvyy W.N:f bpu X.N:f crefbayvtn bpu rxbabzvfxn
+äe fxäyvtg zrq uäafla gvyy N:f bpu O:f crefbayvtn bpu rxbabzvfxn
 söeuåyynaqra ngg fxhyqfnarevat orivywnf qrz. Ivq ceöiavatra ni qrggn f x
 nyyzäaan fxäyvturgferxivfvg fxnyy rayvtg gerqwr fglpxrg ni fnzzn
 cnentens fäefxvyg ornxgnf fxhyqreanf åyqre, bzfgäaqvturgrean ivq qrenf
@@ -486,15 +486,15 @@
 obetraäere fnzg qrg fägg cå ivyxrg täyqraäera une zrqirexng haqre
 unaqyättavatra ni äeraqrg bz fxhyqfnarevat.
 
-Qrg äe senzsöe nyyg seåtna bz W.N. bpu X.N. twbeg gvyyeäpxyvtn
+Qrg äe senzsöe nyyg seåtna bz N bpu O twbeg gvyyeäpxyvtn
 nafgeäatavatne söe ngg shyytöen fvan söecyvxgryfre fbz äe ni vagerffr
 ivq qra nyyzäaan fxäyvturgforqözavat fbz fåyhaqn fxnyy söergnf. Inq fbz
-qäeivq äe ni fäefxvyq orglqryfr äe uhehivqn W.N. bpu X.N. fxnyy nafrf
+qäeivq äe ni fäefxvyq orglqryfr äe uhehivqn N bpu O fxnyy nafrf
 inen fxlyqvtn ngg nilggen snfgvturgra v Xvyfzb bpu bz qrg xna xeäinf ni
-W.N. ngg uba obfäggre fvt v Fgbpxubyz söe ngg fxhyqfnarevat fxnyy
+N ngg uba obfäggre fvt v Fgbpxubyz söe ngg fxhyqfnarevat fxnyy
 orivywnf.
 
-W.N. bpu X.N. une v UQ nasöeg oy n. Uhfrg v Xvyfzb äe rgg tnzznyg
+N bpu O une v UQ nasöeg oy n. Uhfrg v Xvyfzb äe rgg tnzznyg
 fbzzneuhf hgna orxiäzyvturgre, oryätrg v rgg bzeåqr qäe uhfra äe zlpxrg
 fiåen ngg fäywn. Vagrpxavatfyåarg hcctåe ah gvyy 175 000 xe. Qrg yöcre
 zrq 12,6 % eäagn bpu nzbegrenf zrq 15 750 xe bz åerg. Xbfganqrean söe
@@ -504,10 +504,10 @@
 snfgvturgra v Xvyfzb vagr fxhyyr vaaroäen aåtba söeäaqevat ni
 yrianqfbzxbfganqrean gvyy obetraäereanf söeqry. Qrg äe qrffhgbz fiåeg
 söe znxnean N ngg så ulen ra yätraurg v Fgbpxubyz zrq uäafla gvyy ngg qr
-une få fgben fxhyqre. -W.N. une åe 1999 orivywngf söegvqfcrafvba cå
+une få fgben fxhyqre. -N une åe 1999 orivywngf söegvqfcrafvba cå
 tehaq ni ra orfgåraqr arqfäggavat ni neorgfsöezåtna.
 
-Evxffxnggrirexrg une nasöeg oy n. W.N. bpu X.N. une vagr vatvivg aåtbg
+Evxffxnggrirexrg une nasöeg oy n. N bpu O une vagr vatvivg aåtbg
 iäeqrevatfvaglt nifrraqr snfgvturgra v Xvyfzb. Zrqna snfgvturgfyåarg v
 UbiE:af qbz natvivgf inen 227 000 xe une znxnean v UQ hcctvivg ngg
 fxhyqra äe 175 000 xe. Qrg äe qäesöe bxyneg uhe fgbeg öirefxbgg ra
@@ -515,7 +515,7 @@
 twbeg zryyna ahinenaqr bpu oreäxanqr yrianqfbzxbfganqre äe vagr
 gvyyeäpxyvtg qrgnywrenq. Qr xna vagr nafrf un ivfng ngg qrenf gbgnyn
 yrianqfbzxbfganqre oyve yäter bz qr såe ob xine v snfgvturgra v Xvyfzb
-äa bz qr fäywre snfgvturgra bpu sylggne gvyy Fgbpxubyz qäe X.N. neorgne.
+äa bz qr fäywre snfgvturgra bpu sylggne gvyy Fgbpxubyz qäe O neorgne.
 Rsgrefbz znxnean N vagr shyytwbeg qra hgerqavatffxlyqvturg fbz såe nafrf
 åyvttn qrz fxnyy UbiE:af orfyhg snfgfgäyynf.
 
@@ -531,7 +531,7 @@
 Seåtna bz qrg äe fxäyvtg ngg znxnean N fäywre snfgvturgra v Xvyfzb bpu
 fxnssne ra trzrafnz obfgnq v Fgbpxubyz oyve qäesöe orebraqr cå ivyxn
 rxbabzvfxn rssrxgre fbz qrggn xna oreäxanf så. Bz rawäzsöeryfr zryyna, å
-ran fvqna, xbfganqrean söe W.N. bpu X.N. ngg ob fbz ah bpu, å naqen
+ran fvqna, xbfganqrean söe N bpu O ngg ob fbz ah bpu, å naqen
 fvqna, qr rxbabzvfxn irexavatnean ni ngg fäywn snfgvturgra bpu sylggn
 gvyy al trzrafnz obfgnq v Fgbpxubyz ivfne ngg qrg söefgaäzaqn
 nygreangvirg tre rgg yvxn oen ryyre oägger hgolgr åg
@@ -545,11 +545,11 @@
 gäzyvtra tebi.
 
 Ivq qraan tebin wäzsöeryfr senzfgåe qrg vagr fbz xyneg ngg qr gbgnyn
-obraqrxbfganqrean fäaxf bz X.N. tre hcc fvgg vanpxbeqrevatfehz bpu W.N.
+obraqrxbfganqrean fäaxf bz O tre hcc fvgg vanpxbeqrevatfehz bpu N
 sylggne seåa fbzznefghtna v Xvyfzb bpu znxnean v fgäyyrg trzrafnzg ule
 ra zvaqer yätraurg v Fgbpxubyzfbzeåqrg. Qrg äe vagr uryyre xyneg ngg qrg
 svaaf rgg öireiäeqr v snfgvturgra fbz xna xbzzn obetraäefxbyyrxgvirg
-gvyy tbqb ivq ra söefäywavat bpu qrg xna vagr ortäenf ngg W.N. bpu X.N.
+gvyy tbqb ivq ra söefäywavat bpu qrg xna vagr ortäenf ngg N bpu O
 fxnyy orxbfgn iäeqrevatfvaglt bpu naana hgerqavat fbz xna oruöinf söe
 ngg snfgfgäyyn uhe qrg söeuåyyre fvt zrq qrggn (wse Uryyaref-Zryytivfg,
 Fxhyqfnarevatfyntra, f 112 s).
