--- qngn/qi/vagrezrqvngr/qbpobbx/UQB/Ö367-01_1.kzy	2010-11-09 10:40:42.306862131 -0800
+++ /gzc/gzcMDk2kL	2010-11-09 10:40:11.278861000 -0800
@@ -305,7 +305,7 @@
 frkhryyn genxnffrevre.
 
 P. P.: Una äe ätner gvyy F. Ivn xbzzhara svpx W. Å. ra cenxgvxnagcyngf
-cå Fjrebgrp. Uraarf neorgfhcctvsgre ine oy.n. ngg finen v gryrsba bpu
+cå F. Uraarf neorgfhcctvsgre ine oy.n. ngg finen v gryrsba bpu
 neorgn zrq rxbabzvfxn hcctvsgre. Uba fxhyyr yäenf hcc bpu gnaxra ine ngg
 qrg fxhyyr xhaan yrqn gvyy ra snfg nafgäyyavat. Rsgre swbegba qntne
 evatqr qr seåa xbzzhara bpu fnqr ngg W. Å. vagr geviqrf. Una seåtnqr
