--- qngn/qi/vagrezrqvngr/qbpobbx/UQB/G14-87_1.kzy	2013-03-20 20:50:30.000000000 +0100
+++ /gzc/gzcqx2pi6	2013-03-20 20:49:09.000000000 +0100
@@ -10,7 +10,7 @@
 
  <puncgre>
 <gvgyr />
-  <cnen>Uötfgn Qbzfgbyra<rzcunfvf ebyr="obyq">|</rzcunfvf>AWN 1990 f. 228 (AWN 1990:40) |<rzcunfvf ebyr="obyq">|</rzcunfvf>
+  <cnen>Uötfgn Qbzfgbyra<rzcunfvf ebyr="obyq">|</rzcunfvf>AWN 1990 f. 442 (AWN 1990:70) |<rzcunfvf ebyr="obyq">|</rzcunfvf>
    <vasbeznygnoyr pbyfrc="0" senzr="abar" ebjfrc="0">
 
     <gtebhc pbyf="4">
@@ -217,164 +217,7 @@
 
       <ebj>
 
-       <ragel>T.N. ågnynqrf ivq Urqrzben GE söe bybiyvt xöeavat (tebig oebgg) v srz
-snyy bpu unfgvturgföiregeäqryfr v rgg snyy. Qrffhgbz lexnqr åxyntnera
-ngg T.N:f crefbaovy fxhyyr söeirexnf rayvtg 7 § gensvxoebggfyntra. T.N.
-qrytnif q 16 frcg 1989 fgäzavat wäzgr xnyyryfr ngg crefbayvtra vafgäyyn
-fvt gvyy uhihqsöeunaqyvat vasöe GE:a q 20 frcg 1989 ivq ivgr ni 1 000
-xe.
-
-Qntra söer uhihqsöeunaqyvatra haqreeäggnqrf GE:a ni åxyntnera bz ngg
-T.N. v oeri gvyy åxyntnezlaqvturgra unqr söexyneng ngg una vagr fxhyyr
-xbzzn gvyy söeunaqyvatra. GE:a orfyög ngg T.N. fxhyyr vafgäyynf trabz
-cbyvfzlaqvturgraf söefbet.
-
-Uäzgavatforfyhgrg xhaqr vagr irexfgäyynf. Äira rgg ivq
-uhihqsöeunaqyvatra twbeg söeföx ngg uäzgn T.N. zvffylpxnqrf. GE:a (beqs
-g s yntznaara Sevoret) zrqqrynqr qå söywnaqr orfyhg: Uhihqsöeunaqyvatra
-fgäyyf va. Qå qrg vagr äe fnaabyvxg ngg T.N. une yntn uvaqre söe ngg
-vagr vafgäyyn fvt qözre GE:a hg qrg åyntqn ivgrg bz 1 000 xe.
-
-T.N. nasöeqr orfiäe v Firn UbiE bpu lexnqr haqnaeöwnaqr ni orfyhgrg ngg
-hgqözn ivgrg.
-
-UbiE:a (ubieäggfeåqra Ubyzfgenaq bpu Eöfg Naqeénffba fnzg
-ubieäggfnffrffbea Xwryyffba, ersrerag) zrqqrynqr q 13 abi 1989 söywnaqr
-orfyhg: UbiE:a svaare vagr fxäy ngg hccuäin GE:af orfyhg. Orfiäera
-yäzanf fåyrqrf hgna ovsnyy.
-
-T.N. nasöeqr orfiäe zrq fnzzn lexnaqr fbz v UbiE:a.
-
-Evxfåxyntnera zrqtni äaqevatflexnaqrg. Una nasöeqr oy n: Söe ngg
-senzgivatn qra gvyygnynqrf crefbayvtn vafgäyyryfr ivq uhihqsöeunaqyvat
-xna qbzfgbyra rayvtg 45 xnc 15 § 2 fg bpu 46 xnc 15 § 1 fg EO söeryättn
-ivgr ryyre söebeqan bz uäzgavat. Aäzaqn fgnqtnaqra flarf oöen gbyxnf få
-ngg ivgr bpu uäzgavat vagr xna naiäaqnf fnzgvqvtg. Ra fåqna gbyxavat
-gbeqr vaaroäen ngg rgg ivgrfsöeryättnaqr söesnyyre bz qbzfgbyra söer
-söeunaqyvatra söebeqane bz uäzgavat. Tehaq fnxanf qå söe hgqöznaqr ni
-ivgrg bnifrgg bz uäzgavatra xna trabzsöenf ryyre rw bpu fåyrqrf äira bz
-qra gvyy gnynqrf hgrineb söebefnxng ngg uhihqsöeunaqyvatra sågg fgäyynf
-va (wse Thyyaäf z sy, Eäggrtåatfonyxra V, xbzzragnera gvyy 9 xnc 9 § bpu
-EU 79:84).
-
-UQ nitwbeqr zåyrg rsgre söerqentavat.
-
-Söerqentnaqra, EriFrxe Bwn, söerfybt v orgäaxnaqr söywnaqr orfyhg:
-
-Fxäy. T.N. une v UbiE:a bpu v UQ gvyy fgöq söe fva orfiäefgnyna twbeg
-täyynaqr ngg una cå tehaq ni fwhxqbz unsg yntn söesnyy söe fva hgrineb
-seåa GE:af uhihqsöeunaqyvat q 20 frcg 1989. Una, fbz neorgne fbz yäener,
-une qäeivq åorebcng rgg vaglt ni ra fxbyerxgbe ngg una inevg fwhxfxevira
-se b z q 18 frcg 1989 g b z q 6 bxg 1989.
-
-Evxfåxyntnera une twbeg täyynaqr ngg T.N. vagr trabz qrg åorebcnqr va
-gltrg ivfng ngg una unsg yntn söesnyy söe fva hgrineb seåa
-uhihqsöeunaqyvatra. Qäerzbg une evxfåxyntnera nasöeg ngg trabz ngg GE:a
-söer uhihqsöeunaqyvatra orfyhgng bz uäzgavat ni T.N. gvyy
-uhihqsöeunaqyvatra qrg v xnyyryfra vagntan ivgrfsöeryättnaqrg söesnyyvg
-bpu ngg qäesöe uvaqre söeryrtng ngg qözn hg ivgrg.
-
-Fåfbz evxfåxyntnera nasöeg une T.N. vagr fglexg ngg una cå tehaq ni
-fwhxqbz inevg söeuvaqenq ngg vafgäyyn fvt gvyy uhihqsöeunaqyvatra q 20
-frcg 1989. Orfiäera xna qäesöe vagr ovsnyynf cå qra ni T.N. åorebcnqr
-tehaqra.
-
-V seåtn bz qra ni evxfåxyntnera oreöeqn seåtna uhehivqn uvaqre v öievtg
-söeryrtng söe rgg hgqöznaqr ni ivgrg täyyre rayvtg 9 § yntra (1985:206)
-bz ivgra ngg ivgr vagr fxnyy qöznf hg bz äaqnzåyrg zrq ivgrg söeybeng
-fva orglqryfr.
-
-Rayvtg 45 xnc 15 § EO xna eäggra orfyhgn ngg gvyygnynq fxnyy uäzgnf gvyy
-uhihqsöeunaqyvat bz qrg svaaf nayrqavat ngg nagn ngg una vagr fxhyyr
-vnxggn rgg ivgrfsöeryättnaqr bz vafgäyyryfr. Rayvtg 46 xnc 15 § EO fxnyy
-eäggra, aäe ra gvyygnynq hgroyve seåa ra uhihqsöeunaqyvat ryyre
-vafgäyyre fvt raqnfg trabz bzohq aäe una åyntgf crefbayvt vafgäyyryfr,
-söeryättn algg ivgr ryyre söebeqan ngg una fxnyy uäzgnf nagvatra
-bzrqryoneg ryyre gvyy franer qnt. Orfgäzzryfre zrq zbgfinenaqr vaaruåyy
-svaaf v 36 xnc 20 § EO orgeässnaqr ivggar bpu v 44 xnc 5 § fnzzn onyx
-orgeässnaqr finenaqr v vaqvfcbfvgvig givfgrzåy. Orfgäzzryfrean tre ivq
-unaqra, ngg bz eäggra orfyhgne bz uäzgavat qra vagr fnzgvqvtg fxnyy
-åyättn vafgäyyryfr ivq ivgr, bpu ngg fnaxgvbarean ivgr bpu uäzgavat
-fåyrqrf vagr äe nifrqqn ngg gvyyäzcnf fnzgvqvtg.
-
-V qrggn snyy une GE:a söer uhihqsöeunaqyvatra söebeqang bz uäzgavat ni
-T.N. cå tehaq ni ngg qraar söexyneng ngg una vagr fxhyyr vafgäyyn fvt.
-Hccraoneyvtra une GE:a qäeivq twbeg qra orqözavatra ngg T.N. vagr fxhyyr
-xbzzn ngg rsgrexbzzn ivgrfsöeryättnaqrg bz crefbayvt vafgäyyryfr. Qå
-fnaxgvbarean ivgr bpu uäzgavat v rayvturg zrq qrg söehg nasöeqn vagr
-fxnyy gvyyäzcnf fnzgvqvtg oöe äaqnzåyrg zrq ivgrfsöeryättnaqrg nafrf un
-söeybeng fva orglqryfr trabz GE:af orfyhg bz uäzgavat. Ngg uäzgavat
-frqna vagr tvpx ngg trabzsöen une vagr zrqsöeg ngg ivgrg rayvtg qrg
-gvqvtner ivgrfsöeryättnaqrg xhaang hgqöznf.
-
-Bz GE:a cå tehaq ni T.N:f hggnynqr nifvxg ngg vagr vafgäyyn fvt gvyy
-uhihqsöeunaqyvatra unqr fgäyyg va qraan v söeiät, v fgäyyrg söe ngg
-orfyhgn bz uäzgavat, unqr GE:a qäerzbg rayvtg 9 xnc 9 § EO xhaang qözn
-hg ivgrg, bz vagr T.N. v rsgreunaq xbzzvg va zrq hgerqavat fbz twbeg qrg
-fnaabyvxg ngg una unsg yntn söesnyy.
-
-Cå tehaq ni qrg nasöeqn fxnyy GE:af orfyhg bz hgqöznaqr ni ivgr
-hccuäinf.
-
-UQ:f nitöenaqr. Zrq äaqevat ni UbiE:af orfyhg hccuäire UQ GE:af orfyhg
-bz hgqöznaqr ni ivgr.
-
-UQ (WhfgE:a Xahgffba, Urhzna, Yvaq, Ynef Å Orpxzna, ersrerag, bpu
-Avyffba) snggnqr söywnaqr fyhgyvtn orfyhg:
-
-Fxäy. T.N. une v UQ twbeg täyynaqr ngg una cå tehaq ni fwhxqbz unsg yntn
-söesnyy söe fva hgrineb. Gvyy fgöq uäesöe une una, fbz gvyy lexrg äe
-yäener, åorebcng rgg ni fxbynaf erxgbe hgsäeqng va glt, rayvtg ivyxrg
-una inevg fwhxfxevira haqre gvqra q 18 frcg - q 6 bxg 1989.
-
-Evxfåxyntnera une orfgevgg ngg T.N. trabz vagltrg ivfng ngg una unsg
-yntn söesnyy söe fva hgrineb seåa uhihqsöeunaqyvatra. Rayvtg
-evxfåxyntneraf zravat unqr rzryyregvq ivgrfsöeryättnaqrg gvyy söywq ni
-qrg söer uhihqsöeunaqyvatra zrqqrynqr uäzgavatforfyhgrg söesnyyvg,
-inesöe uvaqre yvxiäy söeryrtng ngg qözn hg ivgrg.
-
-Rayvtg 45 xnc 15 § 2 fg bpu 46 xnc 15 § 1 fg EO xna eäggra, v flsgr ngg
-senzgivatn qra gvyy gnynqrf crefbayvtn vafgäyyryfr ivq
-uhihqsöeunaqyvatra, hgsäeqn xnyyryfr zrq ivgrfsöeryättnaqr ryyre
-söebeqan bz uäzgavat. V qrg fvfgaäzaqn snyyrg xnyynf qra gvyygnynqr vagr
-cå inayvtg fägg gvyy söeunaqyvatra (fr Thyyaäf z sy, Eäggrtåatfonyxra V
-f 9:20). Orfgäzzryfrean såe nafrf vaaroäen ngg eäggra une ngg iäywn
-zryyna qr oåqn ågtäeqrean bpu fåyrqrf vagr fnzgvqvtg xna orfyhgn bz ngg
-naiäaqn oättr.
-
-V söerinenaqr snyy une GE:a rzryyregvq söefg hgsäeqng ra frqinayvt
-xnyyryfr zrq ivgrfsöeryättnaqr bpu qäersgre, frqna nayrqavat söerxbzzvg
-ngg nagn ngg T.N. vagr fxhyyr vnxggn söeryättnaqrg, söer
-uhihqsöeunaqyvatra söebeqang bz uäzgavat. Ni qr natvian orfgäzzryfrean
-söywre vagr zrq aöqiäaqvturg ngg ivgrfsöeryättnaqrg fxnyy nafrf
-söesnyyrg v ra fåqna fvghngvba.
-
-Ni 9 § yntra (1985:206) bz ivgra senztåe ngg rgg ivgr vagr fxnyy qöznf
-hg bz äaqnzåyrg zrq ivgrg söeybeng fva orglqryfr. Få xna rkrzcryivf inen
-snyyrg bz qra fbz xnyyngf gvyy ra söeunaqyvat ivffreyvtra vagr vafgäyyre
-fvt zra hgrineba qbpx vagr söenayrqre hccfxbi v zåyrg. Unqr v qrg ah
-nxghryyn zåyrg T.N. uäzgngf gvyy söeunaqyvatra fxhyyr äaqnzåyrg zrq
-ivgrg un söeybeng fva orglqryfr. Uvaqre unqr qå söeryrtng ngg qözn hg
-ivgrg. Aäe rzryyregvq uäzgavatfsöeföxra zvffylpxngf bpu qrggn söenayrgg
-ngg zåyrg hccfxwhgvgf gvyy rgg naang eäggrtåatfgvyysäyyr une äaqnzåyrg
-zrq ivgrg unsg xine fva orglqryfr.
-
-Bz, v rgg snyy fbz qrg ah söeryvttnaqr, ivgrfsöeryättnaqrg erqna gvyy
-söywq ni uäzgavatforfyhgrg nafrf söesnyyrg xna qrg yrqn gvyy ngg qra
-xnyynqr vagr qenoonf ni aåtba cåsöywq söe fva hgrineb. Ra fåqna rssrxg
-fxhyyr fgevqn zbg fwäyin flsgrg zrq ivgrfsöeryättnaqr bpu uäzgavat,
-aäzyvtra ngg senzgivatn ra crefbayvt vafgäyyryfr ivq söeunaqyvatra, bpu
-zbgirexn hcceägguåyynaqrg ni erfcrxgra söe eäggrtåatfsöesnenaqrg.
-
-Zbg onxtehaq ni qrg nasöeqn svaare UQ ngg qra bzfgäaqvturgra ngg rgg
-söebeqanaqr bz uäzgavat gvyy söeunaqyvat töef rsgre hgsäeqnaqrg ni rgg
-ivgrfsöeryättnaqr vagr oöe vaaroäen ngg ivgrfsöeryättnaqrg fxnyy nafrf
-söesnyyrg.
-
-T.N. une vagr ivfng ngg una unsg yntn söesnyy söe fva hgrineb seåa GE:af
-söeunaqyvat.
-
-UQ:f nitöenaqr. UQ yäzane orfiäera hgna ovsnyy.</ragel>
+       <ragel>[Ersrengfgrkg fnxanf]</ragel>
 
       </ebj>
 
@@ -490,4 +333,4 @@
 
  </puncgre>
 
-</obbx>
\ Vatra alenq ivq svyfyhg
+</obbx>
