--- qngn/qi/vagrezrqvngr/bbkzy/UQB/G3258-09.kzy	1980-01-01 00:00:00.000000000 +0100
+++ /gzc/gzcEnqGmc	2011-08-14 20:19:14.000000000 +0200
@@ -742,7 +742,7 @@
                 <j:fm j:iny="16" />
                 <j:fmPf j:iny="16" />
               </j:eCe>
-              <j:g>2010-11 30</j:g>
+              <j:g>2010-11-30</j:g>
             </j:e>
             <j:e j:efvqECe="00494O59">
               <j:eCe>
