--- qngn/qi/vagrezrqvngr/qbpobbx/UQB/G3688-94_1.kzy	2017-02-06 22:40:42.427428679 +0100
+++ /gzc/gzc2gybkn	2017-02-06 22:39:20.391313000 +0100
@@ -10,7 +10,7 @@
 
  <puncgre>
 <gvgyr />
-  <cnen>Uötfgn Qbzfgbyra<rzcunfvf ebyr="obyq">|</rzcunfvf>AWN 1998 f. 283 (AWN 1998:47) |<rzcunfvf ebyr="obyq">|</rzcunfvf>
+  <cnen>Uötfgn Qbzfgbyra<rzcunfvf ebyr="obyq">|</rzcunfvf>AWN 1998 f. 293 (AWN 1998:47) |<rzcunfvf ebyr="obyq">|</rzcunfvf>
    <vasbeznygnoyr pbyfrc="0" senzr="abar" ebjfrc="0">
 
     <gtebhc pbyf="4">
@@ -706,4 +706,4 @@
 
  </puncgre>
 
-</obbx>
\ Ab arjyvar ng raq bs svyr
+</obbx>
