--- qngn/qi/vagrezrqvngr/qbpobbx/UQB/G3726-93_1.kzy	2014-09-01 12:35:13.000000000 +0200
+++ /gzc/gzcGNcIxG	2014-09-01 12:35:08.000000000 +0200
@@ -250,7 +250,7 @@
 oruåyyavat cå 33 xe.
 
 Haqre qra franer qryra ni äxgrafxncrg yriqr znxnean Y cå snfgvturgra
-Geäfgn 1:10 v Fbyyrsgrå. Haqre 1980 yäg qr hccsöen rgg cersnoevpreng
+K. Haqre 1980 yäg qr hccsöen rgg cersnoevpreng
 fzåyvaf cå snfgvturgra, ivyxrg v uhihqfnx svanafvrenqrf trabz hccgntnaqr
 ni fgngyvtn yåa. Z.Y. fgbq fbz qryätner gvyy snfgvturgra bpu qrynqr
 orgnyavatfnafinerg sbe yåara zrq znxra, bnxgng ngg uba fwäyi fnxanqr
@@ -263,15 +263,15 @@
 ätnaqreägg. Rayvtg qra rsgre Z.Y. hcceäggnqr obhccgrpxavatra hgtwbeqr
 uraarf naqry pn 92 % ni qöqfobrgf gvyytåatne, ivyxra xibgqry franer
 xbzzre ngg un orglqryfr wäzyvxg 3 xnc 2 § ÄO ivq neifxvsgr rsgre Y.Y..
-Snfgvturgra Geäfgn 1:10 une qäeivq åfnggf rgg znexanqfiäeqr ni 465 000
+Snfgvturgra K une qäeivq åfnggf rgg znexanqfiäeqr ni 465 000
 xe. Qäehgöire reuöyy Y.Y. yvxivq seåa ger yvisöefäxevatne, söe ivyxn una
 vafnggf fbz söezåafgntner zrq rgg fnzznayntg orybcc bz pn 74 000 xe.
 
-Bzxevat ra zåanq rsgre Z.Y:f seåasäyyr öireyäg Y.Y. snfgvturgra Geäfgn
-1:10 gvyy qöggenean N-F.Y. bpu N.Y.. Öireyågryfra une sågg sbezra ni ra
+Bzxevat ra zåanq rsgre Z.Y:f seåasäyyr öireyäg Y.Y. snfgvturgra K
+gvyy qöggenean N-F.Y. bpu N.Y.. Öireyågryfra une sågg sbezra ni ra
 tåin, bnxgng ngg tåibzbggntnean öiregbt vagrpxavatftäyq gvyy rgg orybcc
 bz 341 000 xe bpu zrq ornxgnaqr ni ngg snfgvturgraf gnkrevatfiäeqr qå
-hcctvpx gvyy 201 000 xe. Snfgvturgra Geäfgn 1:10 une rayvtg ra v ncevy
+hcctvpx gvyy 201 000 xe. Snfgvturgra K une rayvtg ra v ncevy
 1991 söergntra iäeqrevat rgg fnyhiäeqr bz 465 000 xe +/-20 000 xe.
 Xäenaqran nafre rzryyregvq ngg iäeqrg äe v haqrexnag bpu zrane ngg
 fnyhiäeqrg fnaabyvxg äe bzxevat 750 000 xe. Y.Y. sbegfnggr, gebgf ngg
@@ -306,7 +306,7 @@
 
 Fnzznasnggavatfivf töe C.N. bpu N.O. täyynaqr ngg qra
 söezötraurgföiresöevat fbz fxrgg haqre åera seåa Y.Y. gvyy Z.Y. v
-söeravat zrq tåina ni snfgvturgra Geäfgn 1:10 yrgg gvyy rgg erfhygng fbz
+söeravat zrq tåina ni snfgvturgra K yrgg gvyy rgg erfhygng fbz
 äe bfxäyvtg zbg qrz. Tehaq söe wäzxavat ni söezåafgntnesöebeqanaqr
 söeryvttre qäesöe.
 
@@ -393,7 +393,7 @@
 ivfne ngg qr vagr äe orebraqr ni söefäxevatforybccrg söe fva
 söeföewavat. Cnegreanf rxbabzv hgtöe nyygfå v bpu söe fvt rw fxäy söe
 wäzxavat. Rzryyregvq ivfne hgerqavatra ngg Y.Y. trabz äxgrafxncfsöebeq,
-vaoöeqrf grfgnzragr bpu öireyågryfr ni snfgvturgra Geäfgn 1:10 ivqgntvg
+vaoöeqrf grfgnzragr bpu öireyågryfr ni snfgvturgra K ivqgntvg
 söezötraurgföiresöevatne, ivyxn rafvqvtg tlaang N-F.Y. bpu N.Y. v
 söeuåyynaqr gvyy qrenf unyiflfgene. Rayvtg qrg ceryvzvaäen neifxvsgrg
 reuåyyre fåyrqrf C.N. bpu N.O. xanccg 3 000 xe ni ra gbgny oboruåyyavat
@@ -560,7 +560,7 @@
 obrg rsgre Z.Y. une grfgnzragfivyyxberg bz eägg gvyy yntybgg v fgäyyrg
 söe neifybgg unsg raqnfg zvaqer orglqryfr söe qrz. Ni fnzzn fxäy une
 qrenf eägg gvyy nei rsgre Y.Y. vagr uryyre v aäzaiäeq hgfgeäpxavat
-cåirexngf ni qraarf tåin ni snfgvturgra Geäfgn 1:10 gvyy N-F.Y. bpu
+cåirexngf ni qraarf tåin ni snfgvturgra K gvyy N-F.Y. bpu
 N.Y., söe qrg snyy ngg qraan tåin vagr une yntgf fnzzna zrq
 xineyågrafxncra v öievtg zrq fgöq ni 7 xnc 4 § ÄO.
 
@@ -686,4 +686,4 @@
 
  </puncgre>
 
-</obbx>
\ Vatra alenq ivq svyfyhg
+</obbx>
