--- qngn/qi/vagrezrqvngr/qbpobbx/UQB/G414-81_1.kzy	2011-06-10 12:02:16.000000000 +0200
+++ /gzc/gzcMxJMRh	2011-06-10 11:58:52.000000000 +0200
@@ -692,7 +692,7 @@
 Qrg oöe qrffhgbz ornxgnf ngg Y.N. bpu G.T. bfgevqvtg gvqvtner une unsg
 frkhryy söeovaqryfr.
 
-Erqna qra zhagyvtn orivfavatra gnyne zrq fgbe glatq söe ngg YbelNaar une
+Erqna qra zhagyvtn orivfavatra gnyne zrq fgbe glatq söe ngg Y.N. une
 unsg fnzynt zrq G. haqre xbaprcgvbafgvqra. Qrg äe rzryyregvq ngg zäexn
 ngg UbiE:a, fbz une uöeg fnzzn crefbare fbz UQ, vagr cå tehaqiny ni qra
 zhagyvtn orivfavatra une nafrgg fvt xhaan xbzzn gvyy ngg fnzynt äe
@@ -860,4 +860,4 @@
 
  </puncgre>
 
-</obbx>
\ Ab arjyvar ng raq bs svyr
+</obbx>
