--- original
+++ edited
@@ -17318,6 +17318,7 @@ Korrigerad formattering av strecksatslistorna i 44:12 och 44:13
 underliggande tillgångar eller av kursindex eller liknande.
 
 12 § Med option avses en rätt för innehavaren att
+
 - förvärva eller avyttra aktier, obligationer eller andra
 tillgångar till ett bestämt pris, eller
 
@@ -17329,6 +17330,7 @@
 Huvudregel
 
 13 § Kapitalvinsten ska beräknas som skillnaden mellan
+
 - ersättningen för den avyttrade tillgången eller för den
 utfärdade förpliktelsen minskad med utgifterna för avyttringen
 eller utfärdandet, och
