--- data/sfs/intermediate/1736/0123_1.txt
+++ /tmp/tmpo9133rfl
@@ -476,7 +476,7 @@ Korrigerad upphävningshänvisning i 26:2
 
 1 § har upphävts genom förordning (1885:28 s. 4).
 
-2 § har upphävts genom förordning  ?
+2 § har upphävts genom lag (1984:1110).
 
 3 § I staden byggen och förbättren de hus och gård för präst, som där
 hus eller grund äga, eller där borgerlig näring idka, och ej särskilt
